INFO 2023-09-11 15:50:54,867 views 25148 14312 Main page accessed
ERROR 2023-09-11 15:51:33,318 views 25148 18504 Error in about page: division by zero
Traceback (most recent call last):
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\karmaproject\dz_karmaapp\views.py", line 15, in about
    result = 1 / 0
ZeroDivisionError: division by zero
INFO 2023-09-11 15:53:07,839 autoreload 12748 14036 Watching for file changes with StatReloader
INFO 2023-09-11 15:53:09,959 basehttp 12748 19596 "GET /dz/about/ HTTP/1.1" 200 27
INFO 2023-09-11 15:53:22,425 autoreload 12748 14036 C:\Users\karmu\Desktop\STUDIUM\5 - Python\karmaproject\karmaproject\settings.py changed, reloading.
INFO 2023-09-11 15:53:22,768 autoreload 17604 21948 Watching for file changes with StatReloader
INFO 2023-09-11 15:54:31,497 autoreload 17604 21948 C:\Users\karmu\Desktop\STUDIUM\5 - Python\karmaproject\karmaproject\urls.py changed, reloading.
INFO 2023-09-11 15:54:31,843 autoreload 9064 17312 Watching for file changes with StatReloader
INFO 2023-09-11 15:54:35,099 autoreload 9064 17312 C:\Users\karmu\Desktop\STUDIUM\5 - Python\karmaproject\karmaproject\settings.py changed, reloading.
INFO 2023-09-11 15:54:35,437 autoreload 6864 33948 Watching for file changes with StatReloader
INFO 2023-09-11 15:55:24,708 autoreload 6864 33948 C:\Users\karmu\Desktop\STUDIUM\5 - Python\karmaproject\dz_karmaapp\views.py changed, reloading.
INFO 2023-09-11 15:55:25,050 autoreload 19232 20916 Watching for file changes with StatReloader
INFO 2023-09-11 15:56:44,136 autoreload 19232 20916 C:\Users\karmu\Desktop\STUDIUM\5 - Python\karmaproject\karmaproject\settings.py changed, reloading.
INFO 2023-09-11 15:56:44,467 autoreload 30052 14256 Watching for file changes with StatReloader
INFO 2023-09-11 15:57:01,035 autoreload 30052 14256 C:\Users\karmu\Desktop\STUDIUM\5 - Python\karmaproject\karmaproject\settings.py changed, reloading.
INFO 2023-09-11 15:57:01,380 autoreload 33768 19340 Watching for file changes with StatReloader
INFO 2023-09-11 15:58:54,129 autoreload 33768 19340 C:\Users\karmu\Desktop\STUDIUM\5 - Python\karmaproject\dz_karmaapp\views.py changed, reloading.
INFO 2023-09-11 15:58:54,486 autoreload 25520 22828 Watching for file changes with StatReloader
INFO 2023-09-11 15:58:59,867 basehttp 25520 11480 "GET /dz/ HTTP/1.1" 200 35
INFO 2023-09-11 16:01:46,626 autoreload 25520 22828 C:\Users\karmu\Desktop\STUDIUM\5 - Python\karmaproject\dz_karmaapp\views.py changed, reloading.
INFO 2023-09-11 16:01:47,013 autoreload 12252 11324 Watching for file changes with StatReloader
INFO 2023-09-11 16:01:51,106 basehttp 12252 22772 "GET /dz/about/ HTTP/1.1" 200 95
INFO 2023-09-11 16:01:59,547 autoreload 12252 11324 C:\Users\karmu\Desktop\STUDIUM\5 - Python\karmaproject\dz_karmaapp\views.py changed, reloading.
INFO 2023-09-11 16:01:59,921 autoreload 32796 16240 Watching for file changes with StatReloader
INFO 2023-09-11 16:02:02,662 basehttp 32796 25340 "GET /dz/about/ HTTP/1.1" 200 96
INFO 2023-09-11 16:02:04,267 basehttp 32796 25340 "GET /dz/about/ HTTP/1.1" 200 96
INFO 2023-09-11 16:02:06,491 basehttp 32796 25340 "GET /dz/about/ HTTP/1.1" 200 96
INFO 2023-09-11 16:02:30,881 autoreload 32796 16240 C:\Users\karmu\Desktop\STUDIUM\5 - Python\karmaproject\dz_karmaapp\views.py changed, reloading.
INFO 2023-09-11 16:02:31,216 autoreload 20460 10436 Watching for file changes with StatReloader
INFO 2023-09-11 16:02:39,593 autoreload 20460 10436 C:\Users\karmu\Desktop\STUDIUM\5 - Python\karmaproject\dz_karmaapp\views.py changed, reloading.
INFO 2023-09-11 16:02:39,943 autoreload 26356 13980 Watching for file changes with StatReloader
INFO 2023-09-11 16:02:42,683 basehttp 26356 21048 "GET /dz/about/ HTTP/1.1" 200 99
INFO 2023-09-11 16:03:10,908 autoreload 26356 13980 C:\Users\karmu\Desktop\STUDIUM\5 - Python\karmaproject\dz_karmaapp\views.py changed, reloading.
INFO 2023-09-11 16:03:11,313 autoreload 35716 35836 Watching for file changes with StatReloader
INFO 2023-09-11 16:03:12,195 basehttp 35716 6216 "GET /dz/about/ HTTP/1.1" 200 108
INFO 2023-09-11 16:03:34,014 autoreload 35716 35836 C:\Users\karmu\Desktop\STUDIUM\5 - Python\karmaproject\dz_karmaapp\views.py changed, reloading.
INFO 2023-09-11 16:03:34,350 autoreload 36724 12984 Watching for file changes with StatReloader
INFO 2023-09-11 16:03:51,931 autoreload 36724 12984 C:\Users\karmu\Desktop\STUDIUM\5 - Python\karmaproject\dz_karmaapp\views.py changed, reloading.
INFO 2023-09-11 16:03:52,309 autoreload 36392 23960 Watching for file changes with StatReloader
INFO 2023-09-11 16:03:53,950 basehttp 36392 8540 "GET /dz/about/ HTTP/1.1" 200 106
INFO 2023-09-14 15:31:35,174 autoreload 2444 21040 Watching for file changes with StatReloader
INFO 2023-09-14 15:32:39,553 autoreload 15692 20896 Watching for file changes with StatReloader
INFO 2023-09-20 13:45:50,311 autoreload 1956 23516 Watching for file changes with StatReloader
INFO 2023-09-20 13:53:04,785 autoreload 1956 23516 C:\Users\karmu\Desktop\STUDIUM\5 - Python\karmaproject\dz_karmaapp\views.py changed, reloading.
INFO 2023-09-20 13:53:05,170 autoreload 23252 2588 Watching for file changes with StatReloader
INFO 2023-09-20 13:53:16,458 autoreload 23252 2588 C:\Users\karmu\Desktop\STUDIUM\5 - Python\karmaproject\dz_karmaapp\views.py changed, reloading.
INFO 2023-09-20 13:53:16,795 autoreload 23412 24552 Watching for file changes with StatReloader
INFO 2023-09-20 13:53:24,034 autoreload 23412 24552 C:\Users\karmu\Desktop\STUDIUM\5 - Python\karmaproject\dz_karmaapp\views.py changed, reloading.
INFO 2023-09-20 13:53:24,385 autoreload 21992 28248 Watching for file changes with StatReloader
INFO 2023-09-20 13:53:29,097 basehttp 21992 22756 "GET / HTTP/1.1" 200 13
WARNING 2023-09-20 13:53:29,622 log 21992 22756 Not Found: /favicon.ico
WARNING 2023-09-20 13:53:29,623 basehttp 21992 22756 "GET /favicon.ico HTTP/1.1" 404 2547
WARNING 2023-09-20 13:53:35,457 log 21992 22756 Not Found: /client/add
WARNING 2023-09-20 13:53:35,457 basehttp 21992 22756 "GET /client/add HTTP/1.1" 404 2544
INFO 2023-09-20 13:53:49,994 basehttp 21992 22756 "GET /dz/client/add HTTP/1.1" 301 0
ERROR 2023-09-20 13:53:50,025 log 21992 22872 Internal Server Error: /dz/client/add/
Traceback (most recent call last):
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\karmaproject\dz_karmaapp\views.py", line 45, in client_form
    return render(request, 'dz_karmaapp/client_form.html', {'form': form})
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\shortcuts.py", line 24, in render
    content = loader.render_to_string(template_name, context, request, using=using)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\template\loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\template\loader.py", line 19, in get_template
    raise TemplateDoesNotExist(template_name, chain=chain)
django.template.exceptions.TemplateDoesNotExist: dz_karmaapp/client_form.html
ERROR 2023-09-20 13:53:50,026 basehttp 21992 22872 "GET /dz/client/add/ HTTP/1.1" 500 84023
INFO 2023-09-20 13:54:05,058 basehttp 21992 22872 "GET /dz HTTP/1.1" 301 0
INFO 2023-09-20 13:54:05,061 basehttp 21992 14704 "GET /dz/ HTTP/1.1" 200 35
INFO 2023-09-20 13:54:40,390 basehttp 21992 14704 "GET /dz/about HTTP/1.1" 301 0
INFO 2023-09-20 13:54:40,393 basehttp 21992 15552 "GET /dz/about/ HTTP/1.1" 200 106
ERROR 2023-09-20 13:54:46,731 log 21992 15552 Internal Server Error: /dz/client/add/
Traceback (most recent call last):
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\karmaproject\dz_karmaapp\views.py", line 45, in client_form
    return render(request, 'dz_karmaapp/client_form.html', {'form': form})
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\shortcuts.py", line 24, in render
    content = loader.render_to_string(template_name, context, request, using=using)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\template\loader.py", line 61, in render_to_string
    template = get_template(template_name, using=using)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\template\loader.py", line 19, in get_template
    raise TemplateDoesNotExist(template_name, chain=chain)
django.template.exceptions.TemplateDoesNotExist: dz_karmaapp/client_form.html
ERROR 2023-09-20 13:54:46,733 basehttp 21992 15552 "GET /dz/client/add/ HTTP/1.1" 500 84023
ERROR 2023-09-20 13:59:15,597 log 21992 15552 Internal Server Error: /dz/client/add/
Traceback (most recent call last):
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\template\base.py", line 505, in parse
    compile_func = self.tags[command]
KeyError: 'end'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\karmaproject\dz_karmaapp\views.py", line 45, in client_form
    return render(request, 'dz_karmaapp/client_form.html', {'form': form})
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\shortcuts.py", line 24, in render
    content = loader.render_to_string(template_name, context, request, using=using)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\template\loader.py", line 62, in render_to_string
    return template.render(context, request)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\template\backends\django.py", line 61, in render
    return self.template.render(context)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\template\base.py", line 175, in render
    return self._render(context)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\template\base.py", line 167, in _render
    return self.nodelist.render(context)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\template\base.py", line 1005, in render
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\template\base.py", line 1005, in <listcomp>
    return SafeString("".join([node.render_annotated(context) for node in self]))
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\template\base.py", line 966, in render_annotated
    return self.render(context)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\template\loader_tags.py", line 132, in render
    compiled_parent = self.get_parent(context)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\template\loader_tags.py", line 129, in get_parent
    return self.find_template(parent, context)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\template\loader_tags.py", line 107, in find_template
    template, origin = context.template.engine.find_template(
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\template\engine.py", line 157, in find_template
    template = loader.get_template(name, skip=skip)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\template\loaders\cached.py", line 57, in get_template
    template = super().get_template(template_name, skip)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\template\loaders\base.py", line 28, in get_template
    return Template(
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\template\base.py", line 154, in __init__
    self.nodelist = self.compile_nodelist()
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\template\base.py", line 200, in compile_nodelist
    return parser.parse()
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\template\base.py", line 513, in parse
    raise self.error(token, e)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\template\base.py", line 511, in parse
    compiled_result = compile_func(self, token)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\template\loader_tags.py", line 232, in do_block
    nodelist = parser.parse(("endblock",))
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\template\base.py", line 507, in parse
    self.invalid_block_tag(token, command, parse_until)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\template\base.py", line 558, in invalid_block_tag
    raise self.error(
django.template.exceptions.TemplateSyntaxError: Invalid block tag on line 8: 'end', expected 'endblock'. Did you forget to register or load this tag?
ERROR 2023-09-20 13:59:15,601 basehttp 21992 15552 "GET /dz/client/add/ HTTP/1.1" 500 175218
INFO 2023-09-20 13:59:48,475 basehttp 21992 15552 "GET /dz/client/add/ HTTP/1.1" 200 832
INFO 2023-09-20 14:09:21,090 basehttp 21992 20148 "GET /dz/client/add/ HTTP/1.1" 200 1078
ERROR 2023-09-20 14:09:30,724 log 21992 20148 Internal Server Error: /dz/client/add/
Traceback (most recent call last):
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\karmaproject\dz_karmaapp\views.py", line 39, in client_form
    address = form.cleaned_data['address']
KeyError: 'address'
ERROR 2023-09-20 14:09:30,725 basehttp 21992 20148 "POST /dz/client/add/ HTTP/1.1" 500 71957
INFO 2023-09-20 14:09:38,926 basehttp 21992 20148 "GET /dz/client/add/ HTTP/1.1" 200 1078
INFO 2023-09-21 13:34:48,648 autoreload 21992 28248 C:\Users\karmu\Desktop\STUDIUM\5 - Python\karmaproject\dz_karmaapp\forms.py changed, reloading.
INFO 2023-09-21 13:34:49,279 autoreload 23964 23960 Watching for file changes with StatReloader
INFO 2023-09-21 14:06:01,448 autoreload 23964 23960 C:\Users\karmu\Desktop\STUDIUM\5 - Python\karmaproject\dz_karmaapp\models.py changed, reloading.
INFO 2023-09-21 14:06:01,838 autoreload 6184 10992 Watching for file changes with StatReloader
INFO 2023-09-21 14:07:24,496 autoreload 6184 10992 C:\Users\karmu\Desktop\STUDIUM\5 - Python\karmaproject\dz_karmaapp\forms.py changed, reloading.
INFO 2023-09-21 14:07:24,918 autoreload 27348 24976 Watching for file changes with StatReloader
INFO 2023-09-21 14:08:06,814 autoreload 27348 24976 C:\Users\karmu\Desktop\STUDIUM\5 - Python\karmaproject\dz_karmaapp\forms.py changed, reloading.
INFO 2023-09-21 14:08:07,174 autoreload 22976 6580 Watching for file changes with StatReloader
INFO 2023-09-21 15:29:18,391 autoreload 22976 6580 C:\Users\karmu\Desktop\STUDIUM\5 - Python\karmaproject\dz_karmaapp\forms.py changed, reloading.
INFO 2023-09-21 15:29:18,845 autoreload 7360 28800 Watching for file changes with StatReloader
INFO 2023-09-21 15:29:22,016 autoreload 7360 28800 C:\Users\karmu\Desktop\STUDIUM\5 - Python\karmaproject\dz_karmaapp\forms.py changed, reloading.
INFO 2023-09-21 15:29:22,432 autoreload 26496 26480 Watching for file changes with StatReloader
INFO 2023-09-21 16:12:37,693 autoreload 21404 26116 Watching for file changes with StatReloader
INFO 2023-09-21 16:18:07,057 autoreload 21404 26116 C:\Users\karmu\Desktop\STUDIUM\5 - Python\karmaproject\dz_karmaapp\models.py changed, reloading.
INFO 2023-09-21 16:18:07,451 autoreload 12516 29928 Watching for file changes with StatReloader
INFO 2023-09-21 16:18:58,743 autoreload 12516 29928 C:\Users\karmu\Desktop\STUDIUM\5 - Python\karmaproject\dz_karmaapp\models.py changed, reloading.
INFO 2023-09-21 16:18:59,098 autoreload 31244 21056 Watching for file changes with StatReloader
INFO 2023-09-21 16:19:33,000 autoreload 31244 21056 C:\Users\karmu\Desktop\STUDIUM\5 - Python\karmaproject\dz_karmaapp\forms.py changed, reloading.
INFO 2023-09-21 16:19:33,401 autoreload 22868 21904 Watching for file changes with StatReloader
INFO 2023-09-21 16:20:54,238 autoreload 22868 21904 C:\Users\karmu\Desktop\STUDIUM\5 - Python\karmaproject\dz_karmaapp\views.py changed, reloading.
INFO 2023-09-21 16:20:54,609 autoreload 31464 25672 Watching for file changes with StatReloader
INFO 2023-09-21 16:21:07,426 autoreload 600 29360 Watching for file changes with StatReloader
INFO 2023-09-21 16:21:27,900 autoreload 18880 18108 Watching for file changes with StatReloader
INFO 2023-09-21 16:21:37,246 basehttp 18880 27692 "GET /dz/product/add/ HTTP/1.1" 200 1123
INFO 2023-09-21 16:21:58,326 basehttp 18880 27692 "POST /dz/product/add/ HTTP/1.1" 200 1217
INFO 2023-09-21 16:22:10,579 basehttp 18880 27692 "POST /dz/product/add/ HTTP/1.1" 200 1214
INFO 2023-09-21 16:22:20,008 basehttp 18880 27692 "POST /dz/product/add/ HTTP/1.1" 200 1214
INFO 2023-09-21 17:34:06,762 autoreload 18880 18108 C:\Users\karmu\Desktop\STUDIUM\5 - Python\karmaproject\dz_karmaapp\forms.py changed, reloading.
INFO 2023-09-21 17:34:07,211 autoreload 19452 27044 Watching for file changes with StatReloader
INFO 2023-09-21 17:34:13,457 autoreload 19452 27044 C:\Users\karmu\Desktop\STUDIUM\5 - Python\karmaproject\dz_karmaapp\forms.py changed, reloading.
INFO 2023-09-21 17:34:13,851 autoreload 28728 25476 Watching for file changes with StatReloader
INFO 2023-09-21 17:37:56,463 autoreload 28728 25476 C:\Users\karmu\Desktop\STUDIUM\5 - Python\karmaproject\dz_karmaapp\forms.py changed, reloading.
INFO 2023-09-21 17:37:56,837 autoreload 23208 26456 Watching for file changes with StatReloader
INFO 2023-09-21 17:48:43,539 autoreload 23916 21620 Watching for file changes with StatReloader
INFO 2023-09-21 17:48:50,370 basehttp 23916 24368 "GET /dz/product/add/ HTTP/1.1" 200 1123
INFO 2023-09-21 17:49:01,312 basehttp 23916 24368 "POST /dz/product/add/ HTTP/1.1" 200 1217
INFO 2023-09-21 17:49:12,286 basehttp 23916 24368 "POST /dz/product/add/ HTTP/1.1" 200 1219
WARNING 2023-09-21 17:49:23,797 log 23916 24368 Not Found: /dz/user/add
WARNING 2023-09-21 17:49:23,797 basehttp 23916 24368 "GET /dz/user/add HTTP/1.1" 404 3350
INFO 2023-09-21 17:49:27,250 basehttp 23916 24368 "GET /dz/client/add HTTP/1.1" 301 0
INFO 2023-09-21 17:49:27,257 basehttp 23916 18232 "GET /dz/client/add/ HTTP/1.1" 200 1083
ERROR 2023-09-21 17:50:01,867 log 23916 18232 Internal Server Error: /dz/client/add/
Traceback (most recent call last):
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\karmaproject\dz_karmaapp\views.py", line 39, in client_form
    address = form.cleaned_data['address']
KeyError: 'address'
ERROR 2023-09-21 17:50:01,868 basehttp 23916 18232 "POST /dz/client/add/ HTTP/1.1" 500 71863
INFO 2023-09-21 17:51:25,172 autoreload 23916 21620 C:\Users\karmu\Desktop\STUDIUM\5 - Python\karmaproject\dz_karmaapp\migrations\0001_initial.py changed, reloading.
INFO 2023-09-21 17:51:25,538 autoreload 18928 25884 Watching for file changes with StatReloader
INFO 2023-09-21 17:51:45,127 autoreload 18928 25884 C:\Users\karmu\Desktop\STUDIUM\5 - Python\karmaproject\dz_karmaapp\migrations\0001_initial.py changed, reloading.
INFO 2023-09-21 17:51:45,480 autoreload 28848 25672 Watching for file changes with StatReloader
INFO 2023-09-21 17:52:40,753 autoreload 28848 25672 C:\Users\karmu\Desktop\STUDIUM\5 - Python\karmaproject\dz_karmaapp\migrations\0001_initial.py changed, reloading.
INFO 2023-09-21 17:52:41,109 autoreload 26372 31660 Watching for file changes with StatReloader
INFO 2023-09-21 17:54:38,059 autoreload 30332 25580 Watching for file changes with StatReloader
ERROR 2023-09-21 17:54:45,356 log 30332 14580 Internal Server Error: /dz/client/add/
Traceback (most recent call last):
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\karmaproject\dz_karmaapp\views.py", line 39, in client_form
    address = form.cleaned_data['address']
KeyError: 'address'
ERROR 2023-09-21 17:54:45,357 basehttp 30332 14580 "POST /dz/client/add/ HTTP/1.1" 500 71863
INFO 2023-09-21 17:57:36,298 basehttp 30332 14580 "GET /dz/product/add/ HTTP/1.1" 200 1123
INFO 2023-09-21 17:57:42,539 basehttp 30332 14580 "GET /dz/client/add/ HTTP/1.1" 200 1083
INFO 2023-09-21 17:59:27,510 autoreload 30332 25580 C:\Users\karmu\Desktop\STUDIUM\5 - Python\karmaproject\dz_karmaapp\views.py changed, reloading.
INFO 2023-09-21 17:59:27,899 autoreload 25380 21284 Watching for file changes with StatReloader
INFO 2023-09-21 17:59:45,349 autoreload 25380 21284 C:\Users\karmu\Desktop\STUDIUM\5 - Python\karmaproject\dz_karmaapp\views.py changed, reloading.
INFO 2023-09-21 17:59:45,705 autoreload 27104 25324 Watching for file changes with StatReloader
INFO 2023-09-21 18:01:24,746 autoreload 27104 25324 C:\Users\karmu\Desktop\STUDIUM\5 - Python\karmaproject\dz_karmaapp\views.py changed, reloading.
INFO 2023-09-21 18:01:25,128 autoreload 28132 30348 Watching for file changes with StatReloader
INFO 2023-09-21 18:01:57,828 autoreload 28132 30348 C:\Users\karmu\Desktop\STUDIUM\5 - Python\karmaproject\dz_karmaapp\views.py changed, reloading.
INFO 2023-09-21 18:01:58,222 autoreload 24940 15224 Watching for file changes with StatReloader
INFO 2023-09-21 18:02:28,905 autoreload 24940 15224 C:\Users\karmu\Desktop\STUDIUM\5 - Python\karmaproject\dz_karmaapp\views.py changed, reloading.
INFO 2023-09-21 18:02:29,276 autoreload 15880 11076 Watching for file changes with StatReloader
INFO 2023-09-21 18:03:40,787 autoreload 15880 11076 C:\Users\karmu\Desktop\STUDIUM\5 - Python\karmaproject\dz_karmaapp\views.py changed, reloading.
INFO 2023-09-21 18:03:41,177 autoreload 27356 28192 Watching for file changes with StatReloader
INFO 2023-09-21 18:03:46,648 basehttp 27356 6404 "GET /dz/client/add/ HTTP/1.1" 200 1065
ERROR 2023-09-21 18:04:14,258 log 27356 6404 Internal Server Error: /dz/client/add/
Traceback (most recent call last):
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\karmaproject\dz_karmaapp\views.py", line 40, in client_form
    address = form.cleaned_data['address']
KeyError: 'address'
ERROR 2023-09-21 18:04:14,259 basehttp 27356 6404 "POST /dz/client/add/ HTTP/1.1" 500 72095
INFO 2023-09-21 18:05:30,066 autoreload 27356 28192 C:\Users\karmu\Desktop\STUDIUM\5 - Python\karmaproject\dz_karmaapp\models.py changed, reloading.
INFO 2023-09-21 18:05:30,456 autoreload 11116 22828 Watching for file changes with StatReloader
INFO 2023-09-21 18:05:32,953 basehttp 11116 27288 "GET /dz/client/add/ HTTP/1.1" 200 1065
INFO 2023-09-21 18:05:34,256 basehttp 11116 27288 "GET /dz/client/add/ HTTP/1.1" 200 1065
INFO 2023-09-21 18:05:44,036 autoreload 11116 22828 C:\Users\karmu\Desktop\STUDIUM\5 - Python\karmaproject\dz_karmaapp\models.py changed, reloading.
INFO 2023-09-21 18:05:44,399 autoreload 26236 18508 Watching for file changes with StatReloader
INFO 2023-09-21 18:05:47,681 autoreload 26236 18508 C:\Users\karmu\Desktop\STUDIUM\5 - Python\karmaproject\dz_karmaapp\models.py changed, reloading.
INFO 2023-09-21 18:05:48,043 autoreload 8124 21972 Watching for file changes with StatReloader
INFO 2023-09-21 18:05:52,302 autoreload 8124 21972 C:\Users\karmu\Desktop\STUDIUM\5 - Python\karmaproject\dz_karmaapp\models.py changed, reloading.
INFO 2023-09-21 18:05:52,669 autoreload 28236 13108 Watching for file changes with StatReloader
INFO 2023-09-21 18:06:02,038 autoreload 28236 13108 C:\Users\karmu\Desktop\STUDIUM\5 - Python\karmaproject\dz_karmaapp\models.py changed, reloading.
INFO 2023-09-21 18:06:02,440 autoreload 24968 28324 Watching for file changes with StatReloader
INFO 2023-09-21 18:07:10,100 autoreload 24968 28324 C:\Users\karmu\Desktop\STUDIUM\5 - Python\karmaproject\dz_karmaapp\forms.py changed, reloading.
INFO 2023-09-21 18:07:10,455 autoreload 26668 12568 Watching for file changes with StatReloader
INFO 2023-09-21 18:07:21,874 autoreload 26668 12568 C:\Users\karmu\Desktop\STUDIUM\5 - Python\karmaproject\dz_karmaapp\forms.py changed, reloading.
INFO 2023-09-21 18:07:22,235 autoreload 24960 23860 Watching for file changes with StatReloader
INFO 2023-09-21 18:07:39,357 basehttp 24960 6264 "GET /dz/client/add/ HTTP/1.1" 200 1065
INFO 2023-09-21 18:07:40,718 basehttp 24960 6264 "GET /dz/client/add/ HTTP/1.1" 200 1065
INFO 2023-09-21 18:07:41,418 basehttp 24960 6264 "GET /dz/client/add/ HTTP/1.1" 200 1065
INFO 2023-09-21 18:07:41,622 basehttp 24960 6264 "GET /dz/client/add/ HTTP/1.1" 200 1065
INFO 2023-09-21 18:07:41,800 basehttp 24960 6264 "GET /dz/client/add/ HTTP/1.1" 200 1065
INFO 2023-09-21 18:07:41,984 basehttp 24960 6264 "GET /dz/client/add/ HTTP/1.1" 200 1065
INFO 2023-09-21 18:07:42,161 basehttp 24960 6264 "GET /dz/client/add/ HTTP/1.1" 200 1065
INFO 2023-09-21 18:07:47,294 basehttp 24960 6264 "GET /dz/product/add/ HTTP/1.1" 200 1123
INFO 2023-09-21 18:16:08,948 autoreload 24960 23860 C:\Users\karmu\Desktop\STUDIUM\5 - Python\karmaproject\dz_karmaapp\models.py changed, reloading.
INFO 2023-09-21 18:16:09,298 autoreload 24312 29512 Watching for file changes with StatReloader
INFO 2023-09-21 18:16:17,243 basehttp 24312 24216 "GET /dz/client/add/ HTTP/1.1" 200 1065
INFO 2023-09-21 18:16:27,018 basehttp 24312 24216 "GET /dz/client/add/ HTTP/1.1" 200 1065
INFO 2023-09-21 18:16:31,930 basehttp 24312 24216 "GET /dz/product/add/ HTTP/1.1" 200 1123
INFO 2023-09-21 18:21:38,429 autoreload 24312 29512 C:\Users\karmu\Desktop\STUDIUM\5 - Python\karmaproject\karmaproject\settings.py changed, reloading.
INFO 2023-09-21 18:21:38,786 autoreload 22800 16296 Watching for file changes with StatReloader
INFO 2023-09-21 18:22:52,604 autoreload 22800 16296 C:\Users\karmu\Desktop\STUDIUM\5 - Python\karmaproject\dz_karmaapp\views.py changed, reloading.
INFO 2023-09-21 18:22:52,961 autoreload 16768 11492 Watching for file changes with StatReloader
INFO 2023-09-21 18:23:41,220 autoreload 16768 11492 C:\Users\karmu\Desktop\STUDIUM\5 - Python\karmaproject\dz_karmaapp\views.py changed, reloading.
INFO 2023-09-21 18:23:41,625 autoreload 28312 22864 Watching for file changes with StatReloader
INFO 2023-09-21 18:24:00,261 autoreload 28312 22864 C:\Users\karmu\Desktop\STUDIUM\5 - Python\karmaproject\dz_karmaapp\views.py changed, reloading.
INFO 2023-09-21 18:24:00,653 autoreload 12672 13396 Watching for file changes with StatReloader
INFO 2023-09-21 18:28:06,210 autoreload 12672 13396 C:\Users\karmu\Desktop\STUDIUM\5 - Python\karmaproject\dz_karmaapp\views.py changed, reloading.
INFO 2023-09-21 18:28:06,633 autoreload 16232 23720 Watching for file changes with StatReloader
INFO 2023-09-21 18:28:53,849 autoreload 16232 23720 C:\Users\karmu\Desktop\STUDIUM\5 - Python\karmaproject\dz_karmaapp\views.py changed, reloading.
INFO 2023-09-21 18:28:54,244 autoreload 27600 30300 Watching for file changes with StatReloader
INFO 2023-09-21 18:29:25,934 basehttp 27600 29500 "GET /dz/client/add/ HTTP/1.1" 200 1065
INFO 2023-09-21 18:29:28,118 basehttp 27600 29500 "GET /dz/product/add/ HTTP/1.1" 200 1196
ERROR 2023-09-21 18:29:40,115 log 27600 29500 Internal Server Error: /dz/product/add/
Traceback (most recent call last):
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\karmaproject\dz_karmaapp\views.py", line 57, in product_form
    description = form.cleaned_data['description']
KeyError: 'description'
ERROR 2023-09-21 18:29:40,117 basehttp 27600 29500 "POST /dz/product/add/ HTTP/1.1" 500 72236
ERROR 2023-09-21 18:30:38,518 log 27600 29500 Internal Server Error: /dz/product/add/
Traceback (most recent call last):
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\karmaproject\dz_karmaapp\views.py", line 57, in product_form
    description = form.cleaned_data['description']
KeyError: 'description'
ERROR 2023-09-21 18:30:38,519 basehttp 27600 29500 "POST /dz/product/add/ HTTP/1.1" 500 72236
INFO 2023-09-21 18:31:46,618 autoreload 27600 30300 C:\Users\karmu\Desktop\STUDIUM\5 - Python\karmaproject\dz_karmaapp\forms.py changed, reloading.
INFO 2023-09-21 18:31:46,985 autoreload 21928 16200 Watching for file changes with StatReloader
INFO 2023-09-21 18:31:52,501 basehttp 21928 15352 "GET /dz/product/add/ HTTP/1.1" 200 1348
INFO 2023-09-21 18:33:11,151 autoreload 21928 16200 C:\Users\karmu\Desktop\STUDIUM\5 - Python\karmaproject\dz_karmaapp\forms.py changed, reloading.
INFO 2023-09-21 18:33:11,534 autoreload 14964 16184 Watching for file changes with StatReloader
INFO 2023-09-21 18:33:32,176 autoreload 14964 16184 C:\Users\karmu\Desktop\STUDIUM\5 - Python\karmaproject\dz_karmaapp\models.py changed, reloading.
INFO 2023-09-21 18:33:32,554 autoreload 23020 24464 Watching for file changes with StatReloader
INFO 2023-09-21 18:36:13,667 autoreload 23020 24464 C:\Users\karmu\Desktop\STUDIUM\5 - Python\karmaproject\dz_karmaapp\models.py changed, reloading.
INFO 2023-09-21 18:36:14,065 autoreload 30368 29896 Watching for file changes with StatReloader
INFO 2023-09-21 18:37:12,515 autoreload 30368 29896 C:\Users\karmu\Desktop\STUDIUM\5 - Python\karmaproject\dz_karmaapp\forms.py changed, reloading.
INFO 2023-09-21 18:37:12,867 autoreload 6560 19680 Watching for file changes with StatReloader
INFO 2023-09-21 18:37:21,385 basehttp 6560 14676 "GET /dz/product/add/ HTTP/1.1" 200 1371
INFO 2023-09-21 18:37:33,316 basehttp 6560 14676 "POST /dz/product/add/ HTTP/1.1" 200 1416
INFO 2023-09-21 18:37:37,496 basehttp 6560 14676 "GET /dz/client/add/ HTTP/1.1" 200 1245
INFO 2023-09-21 18:38:06,209 basehttp 6560 14676 "POST /dz/client/add/ HTTP/1.1" 200 1321
INFO 2023-09-21 18:43:12,250 autoreload 6560 19680 C:\Users\karmu\Desktop\STUDIUM\5 - Python\karmaproject\dz_karmaapp\views.py changed, reloading.
INFO 2023-09-21 18:43:12,659 autoreload 16416 3524 Watching for file changes with StatReloader
INFO 2023-09-23 10:02:29,464 autoreload 14428 19580 Watching for file changes with StatReloader
INFO 2023-09-23 10:02:40,843 basehttp 14428 16936 "GET /admin HTTP/1.1" 301 0
INFO 2023-09-23 10:02:40,851 basehttp 14428 18780 "GET /admin/ HTTP/1.1" 302 0
INFO 2023-09-23 10:02:40,884 basehttp 14428 18780 "GET /admin/login/?next=/admin/ HTTP/1.1" 200 4267
INFO 2023-09-23 10:02:41,015 basehttp 14428 18780 "GET /static/admin/css/base.css HTTP/1.1" 200 21207
INFO 2023-09-23 10:02:41,017 basehttp 14428 18780 "GET /static/admin/js/nav_sidebar.js HTTP/1.1" 200 3063
INFO 2023-09-23 10:02:41,038 basehttp 14428 17844 "GET /static/admin/css/dark_mode.css HTTP/1.1" 200 2929
INFO 2023-09-23 10:02:41,042 basehttp 14428 16596 "GET /static/admin/css/login.css HTTP/1.1" 200 958
INFO 2023-09-23 10:02:41,043 basehttp 14428 4628 "GET /static/admin/css/nav_sidebar.css HTTP/1.1" 200 2694
INFO 2023-09-23 10:02:41,043 basehttp 14428 8800 "GET /static/admin/css/responsive.css HTTP/1.1" 200 18533
INFO 2023-09-23 10:02:41,045 basehttp 14428 4224 "GET /static/admin/js/theme.js HTTP/1.1" 200 1943
WARNING 2023-09-23 10:02:41,494 log 14428 4224 Not Found: /favicon.ico
WARNING 2023-09-23 10:02:41,495 basehttp 14428 4224 "GET /favicon.ico HTTP/1.1" 404 2547
INFO 2023-09-23 10:02:45,478 basehttp 14428 4224 "POST /admin/login/?next=/admin/ HTTP/1.1" 302 0
INFO 2023-09-23 10:02:45,490 basehttp 14428 4224 "GET /admin/ HTTP/1.1" 200 5852
INFO 2023-09-23 10:02:45,500 basehttp 14428 4224 "GET /static/admin/css/dashboard.css HTTP/1.1" 200 441
INFO 2023-09-23 10:02:45,510 basehttp 14428 8800 "GET /static/admin/img/icon-changelink.svg HTTP/1.1" 200 380
INFO 2023-09-23 10:02:45,510 basehttp 14428 4224 "GET /static/admin/img/icon-addlink.svg HTTP/1.1" 200 331
INFO 2023-09-23 10:02:51,473 basehttp 14428 8800 "GET /admin/auth/user/add/ HTTP/1.1" 200 10988
INFO 2023-09-23 10:02:51,488 basehttp 14428 8800 "GET /static/admin/css/forms.css HTTP/1.1" 200 9047
INFO 2023-09-23 10:02:51,488 basehttp 14428 16596 "GET /static/admin/js/jquery.init.js HTTP/1.1" 200 347
INFO 2023-09-23 10:02:51,489 basehttp 14428 18780 "GET /static/admin/js/admin/RelatedObjectLookups.js HTTP/1.1" 200 8943
INFO 2023-09-23 10:02:51,490 basehttp 14428 17844 "GET /static/admin/js/core.js HTTP/1.1" 200 5682
INFO 2023-09-23 10:02:51,497 basehttp 14428 4224 "GET /admin/jsi18n/ HTTP/1.1" 200 16066
INFO 2023-09-23 10:02:51,497 basehttp 14428 8800 "GET /static/admin/js/actions.js HTTP/1.1" 200 7872
INFO 2023-09-23 10:02:51,499 basehttp 14428 18780 "GET /static/admin/js/prepopulate.js HTTP/1.1" 200 1531
INFO 2023-09-23 10:02:51,499 basehttp 14428 16596 "GET /static/admin/js/urlify.js HTTP/1.1" 200 7887
INFO 2023-09-23 10:02:51,500 basehttp 14428 8800 "GET /static/admin/js/prepopulate_init.js HTTP/1.1" 200 586
INFO 2023-09-23 10:02:51,501 basehttp 14428 4224 "GET /static/admin/css/widgets.css HTTP/1.1" 200 11900
INFO 2023-09-23 10:02:51,501 basehttp 14428 4628 "GET /static/admin/js/vendor/jquery/jquery.js HTTP/1.1" 200 292458
INFO 2023-09-23 10:02:51,502 basehttp 14428 17844 "GET /static/admin/js/vendor/xregexp/xregexp.js HTTP/1.1" 200 232381
INFO 2023-09-23 10:02:51,505 basehttp 14428 17844 "GET /static/admin/js/change_form.js HTTP/1.1" 200 606
INFO 2023-09-23 10:03:18,647 basehttp 14428 17844 "POST /admin/auth/user/add/ HTTP/1.1" 200 11265
INFO 2023-09-23 10:03:18,660 basehttp 14428 17844 "GET /admin/jsi18n/ HTTP/1.1" 200 16066
INFO 2023-09-23 10:03:34,693 basehttp 14428 17844 "POST /admin/auth/user/add/ HTTP/1.1" 200 11156
INFO 2023-09-23 10:03:34,706 basehttp 14428 17844 "GET /admin/jsi18n/ HTTP/1.1" 200 16066
INFO 2023-09-23 10:03:51,209 basehttp 14428 17844 "POST /admin/auth/user/add/ HTTP/1.1" 302 0
INFO 2023-09-23 10:03:51,253 basehttp 14428 17844 "GET /admin/auth/user/2/change/ HTTP/1.1" 200 22734
INFO 2023-09-23 10:03:51,269 basehttp 14428 4224 "GET /static/admin/js/SelectBox.js HTTP/1.1" 200 4530
INFO 2023-09-23 10:03:51,269 basehttp 14428 8800 "GET /static/admin/js/SelectFilter2.js HTTP/1.1" 200 15292
INFO 2023-09-23 10:03:51,269 basehttp 14428 4628 "GET /static/admin/js/admin/DateTimeShortcuts.js HTTP/1.1" 200 19319
INFO 2023-09-23 10:03:51,269 basehttp 14428 17844 "GET /static/admin/js/calendar.js HTTP/1.1" 200 8466
INFO 2023-09-23 10:03:51,272 basehttp 14428 16596 "GET /admin/jsi18n/ HTTP/1.1" 200 16066
INFO 2023-09-23 10:03:51,290 basehttp 14428 16596 "GET /static/admin/img/icon-yes.svg HTTP/1.1" 200 436
INFO 2023-09-23 10:03:51,387 basehttp 14428 4224 "GET /static/admin/img/icon-calendar.svg HTTP/1.1" 200 1086
INFO 2023-09-23 10:03:51,388 basehttp 14428 18780 "GET /static/admin/img/icon-clock.svg HTTP/1.1" 200 677
INFO 2023-09-23 10:03:51,389 basehttp 14428 17844 "GET /static/admin/img/search.svg HTTP/1.1" 200 458
INFO 2023-09-23 10:03:51,389 basehttp 14428 16596 "GET /static/admin/img/icon-unknown.svg HTTP/1.1" 200 655
INFO 2023-09-23 10:03:51,389 basehttp 14428 8800 "GET /static/admin/img/selector-icons.svg HTTP/1.1" 200 3291
INFO 2023-09-23 10:03:51,389 basehttp 14428 4628 "GET /static/admin/img/icon-unknown-alt.svg HTTP/1.1" 200 655
INFO 2023-09-23 10:03:54,867 basehttp 14428 8800 "GET /admin/auth/group/add/ HTTP/1.1" 200 12123
INFO 2023-09-23 10:03:54,882 basehttp 14428 8800 "GET /admin/jsi18n/ HTTP/1.1" 200 16066
INFO 2023-09-23 10:04:52,554 basehttp 14428 8800 "POST /admin/auth/group/add/ HTTP/1.1" 302 0
INFO 2023-09-23 10:04:52,573 basehttp 14428 8800 "GET /admin/auth/group/ HTTP/1.1" 200 9120
INFO 2023-09-23 10:04:52,586 basehttp 14428 8800 "GET /static/admin/css/changelists.css HTTP/1.1" 200 6566
INFO 2023-09-23 10:04:52,588 basehttp 14428 4628 "GET /admin/jsi18n/ HTTP/1.1" 200 16066
INFO 2023-09-23 10:04:52,595 basehttp 14428 4628 "GET /static/admin/js/filters.js HTTP/1.1" 200 978
INFO 2023-09-23 10:04:52,602 basehttp 14428 4628 "GET /static/admin/img/tooltag-add.svg HTTP/1.1" 200 331
INFO 2023-09-23 10:04:54,864 basehttp 14428 4628 "GET /admin/auth/user/ HTTP/1.1" 200 11997
INFO 2023-09-23 10:04:54,878 basehttp 14428 4628 "GET /static/admin/img/icon-no.svg HTTP/1.1" 200 560
INFO 2023-09-23 10:04:54,881 basehttp 14428 8800 "GET /admin/jsi18n/ HTTP/1.1" 200 16066
INFO 2023-09-23 10:04:54,895 basehttp 14428 8800 "GET /static/admin/img/sorting-icons.svg HTTP/1.1" 200 1097
INFO 2023-09-23 10:04:58,331 basehttp 14428 8800 "GET /admin/auth/user/2/change/ HTTP/1.1" 200 22481
INFO 2023-09-23 10:04:58,346 basehttp 14428 8800 "GET /admin/jsi18n/ HTTP/1.1" 200 16066
INFO 2023-09-23 10:05:27,777 basehttp 14428 8800 "POST /admin/auth/user/2/change/ HTTP/1.1" 302 0
INFO 2023-09-23 10:05:27,789 basehttp 14428 8800 "GET /admin/auth/user/ HTTP/1.1" 200 12220
INFO 2023-09-23 10:05:27,804 basehttp 14428 8800 "GET /admin/jsi18n/ HTTP/1.1" 200 16066
INFO 2023-09-23 10:11:22,501 autoreload 14428 19580 C:\Users\karmu\Desktop\STUDIUM\5 - Python\karmaproject\dz_karmaapp\models.py changed, reloading.
INFO 2023-09-23 10:11:22,888 autoreload 3492 10860 Watching for file changes with StatReloader
INFO 2023-09-23 10:12:51,028 autoreload 3492 10860 C:\Users\karmu\Desktop\STUDIUM\5 - Python\karmaproject\dz_karmaapp\models.py changed, reloading.
INFO 2023-09-23 10:12:51,369 autoreload 22136 15016 Watching for file changes with StatReloader
INFO 2023-09-23 10:16:41,661 autoreload 18324 12604 Watching for file changes with StatReloader
INFO 2023-09-23 10:16:47,936 basehttp 18324 18664 "GET /admin/ HTTP/1.1" 200 7987
INFO 2023-09-23 10:21:42,192 basehttp 18324 18664 "GET /admin/ HTTP/1.1" 200 7987
INFO 2023-09-23 10:32:06,204 autoreload 18324 12604 C:\Users\karmu\Desktop\STUDIUM\5 - Python\karmaproject\dz_karmaapp\models.py changed, reloading.
INFO 2023-09-23 10:32:06,572 autoreload 21424 22068 Watching for file changes with StatReloader
INFO 2023-09-23 10:32:41,412 autoreload 21424 22068 C:\Users\karmu\Desktop\STUDIUM\5 - Python\karmaproject\dz_karmaapp\models.py changed, reloading.
INFO 2023-09-23 10:32:41,764 autoreload 8540 15220 Watching for file changes with StatReloader
INFO 2023-09-23 11:32:13,400 autoreload 5940 2832 Watching for file changes with StatReloader
INFO 2023-09-23 11:34:58,477 autoreload 14736 23504 Watching for file changes with StatReloader
INFO 2023-09-23 11:35:10,055 basehttp 14736 12712 "GET /admin/ HTTP/1.1" 302 0
INFO 2023-09-23 11:35:10,082 basehttp 14736 12712 "GET /admin/login/?next=/admin/ HTTP/1.1" 200 4267
INFO 2023-09-23 11:35:10,120 basehttp 14736 12712 "GET /static/admin/css/login.css HTTP/1.1" 200 958
INFO 2023-09-23 11:35:24,728 basehttp 14736 12712 "POST /admin/login/?next=/admin/ HTTP/1.1" 200 4534
INFO 2023-09-23 11:35:34,428 basehttp 14736 12712 "POST /admin/login/?next=/admin/ HTTP/1.1" 200 4534
INFO 2023-09-23 11:35:41,876 basehttp 14736 12712 "POST /admin/login/?next=/admin/ HTTP/1.1" 200 4534
INFO 2023-09-23 11:37:38,258 autoreload 14184 20332 Watching for file changes with StatReloader
INFO 2023-09-23 11:37:43,659 basehttp 14184 16468 "POST /admin/login/?next=/admin/ HTTP/1.1" 302 0
INFO 2023-09-23 11:37:43,691 basehttp 14184 16468 "GET /admin/ HTTP/1.1" 200 7042
INFO 2023-09-23 11:37:48,611 basehttp 14184 16468 "GET /admin/auth/user/add/ HTTP/1.1" 200 11885
INFO 2023-09-23 11:37:48,735 basehttp 14184 21792 "GET /static/admin/js/actions.js HTTP/1.1" 200 7872
INFO 2023-09-23 11:37:48,737 basehttp 14184 21792 "GET /static/admin/js/prepopulate.js HTTP/1.1" 200 1531
INFO 2023-09-23 11:37:48,739 basehttp 14184 13996 "GET /static/admin/js/urlify.js HTTP/1.1" 200 7887
INFO 2023-09-23 11:37:48,740 basehttp 14184 21792 "GET /static/admin/js/vendor/xregexp/xregexp.js HTTP/1.1" 200 232381
INFO 2023-09-23 11:37:48,746 basehttp 14184 10092 "GET /static/admin/js/admin/RelatedObjectLookups.js HTTP/1.1" 200 8943
INFO 2023-09-23 11:37:48,749 basehttp 14184 13996 "GET /admin/jsi18n/ HTTP/1.1" 200 16066
INFO 2023-09-23 11:37:48,749 basehttp 14184 3792 "GET /static/admin/js/core.js HTTP/1.1" 200 5682
INFO 2023-09-23 11:37:48,750 basehttp 14184 10060 "GET /static/admin/js/jquery.init.js HTTP/1.1" 200 347
INFO 2023-09-23 11:37:48,751 basehttp 14184 16468 "GET /static/admin/js/vendor/jquery/jquery.js HTTP/1.1" 200 292458
INFO 2023-09-23 11:38:03,912 basehttp 14184 16468 "POST /admin/auth/user/add/ HTTP/1.1" 302 0
INFO 2023-09-23 11:38:03,947 basehttp 14184 16468 "GET /admin/auth/user/2/change/ HTTP/1.1" 200 23930
INFO 2023-09-23 11:38:03,960 basehttp 14184 16468 "GET /admin/jsi18n/ HTTP/1.1" 200 16066
INFO 2023-09-23 11:38:04,049 basehttp 14184 16468 "GET /static/admin/img/search.svg HTTP/1.1" 200 458
INFO 2023-09-23 11:38:08,400 basehttp 14184 16468 "GET /admin/auth/group/add/ HTTP/1.1" 200 13319
INFO 2023-09-23 11:38:08,413 basehttp 14184 16468 "GET /admin/jsi18n/ HTTP/1.1" 200 16066
INFO 2023-09-23 11:38:16,309 basehttp 14184 16468 "GET /admin/dz_karmaapp/product/add/ HTTP/1.1" 200 13338
INFO 2023-09-23 11:38:16,322 basehttp 14184 10060 "GET /static/admin/img/icon-viewlink.svg HTTP/1.1" 200 581
INFO 2023-09-23 11:38:16,324 basehttp 14184 16468 "GET /admin/jsi18n/ HTTP/1.1" 200 16066
INFO 2023-09-23 11:38:54,495 basehttp 14184 16468 "GET /admin/dz_karmaapp/category/add/?_to_field=id&_popup=1 HTTP/1.1" 200 4483
INFO 2023-09-23 11:38:54,508 basehttp 14184 16468 "GET /admin/jsi18n/ HTTP/1.1" 200 16066
INFO 2023-09-23 11:39:01,587 basehttp 14184 16468 "POST /admin/dz_karmaapp/category/add/?_to_field=id&_popup=1 HTTP/1.1" 200 374
INFO 2023-09-23 11:39:01,596 basehttp 14184 16468 "GET /static/admin/js/popup_response.js HTTP/1.1" 200 551
INFO 2023-09-23 11:39:05,173 basehttp 14184 16468 "POST /admin/dz_karmaapp/product/add/ HTTP/1.1" 302 0
INFO 2023-09-23 11:39:05,185 basehttp 14184 16468 "GET /admin/dz_karmaapp/product/add/ HTTP/1.1" 200 13706
INFO 2023-09-23 11:39:05,198 basehttp 14184 16468 "GET /admin/jsi18n/ HTTP/1.1" 200 16066
INFO 2023-09-23 11:39:27,313 basehttp 14184 16468 "POST /admin/dz_karmaapp/product/add/ HTTP/1.1" 302 0
INFO 2023-09-23 11:39:27,331 basehttp 14184 16468 "GET /admin/dz_karmaapp/product/ HTTP/1.1" 200 9997
INFO 2023-09-23 11:39:27,346 basehttp 14184 16468 "GET /admin/jsi18n/ HTTP/1.1" 200 16066
INFO 2023-09-23 11:39:33,480 basehttp 14184 16468 "GET /admin/dz_karmaapp/product/add/ HTTP/1.1" 200 13382
INFO 2023-09-23 11:39:33,494 basehttp 14184 16468 "GET /admin/jsi18n/ HTTP/1.1" 200 16066
INFO 2023-09-23 11:39:58,957 basehttp 14184 16468 "GET /admin/dz_karmaapp/category/add/?_to_field=id&_popup=1 HTTP/1.1" 200 4483
INFO 2023-09-23 11:39:58,971 basehttp 14184 16468 "GET /admin/jsi18n/ HTTP/1.1" 200 16066
INFO 2023-09-23 11:40:01,524 basehttp 14184 16468 "POST /admin/dz_karmaapp/category/add/?_to_field=id&_popup=1 HTTP/1.1" 200 367
INFO 2023-09-23 11:40:04,205 basehttp 14184 16468 "POST /admin/dz_karmaapp/product/add/ HTTP/1.1" 302 0
INFO 2023-09-23 11:40:04,221 basehttp 14184 16468 "GET /admin/dz_karmaapp/product/add/ HTTP/1.1" 200 13747
INFO 2023-09-23 11:40:04,238 basehttp 14184 16468 "GET /admin/jsi18n/ HTTP/1.1" 200 16066
INFO 2023-09-23 11:41:07,112 basehttp 14184 16468 "GET /admin/dz_karmaapp/category/add/?_to_field=id&_popup=1 HTTP/1.1" 200 4483
INFO 2023-09-23 11:41:07,125 basehttp 14184 16468 "GET /admin/jsi18n/ HTTP/1.1" 200 16066
INFO 2023-09-23 11:41:12,512 basehttp 14184 16468 "POST /admin/dz_karmaapp/category/add/?_to_field=id&_popup=1 HTTP/1.1" 200 371
INFO 2023-09-23 11:41:15,112 basehttp 14184 16468 "POST /admin/dz_karmaapp/product/add/ HTTP/1.1" 302 0
INFO 2023-09-23 11:41:15,124 basehttp 14184 16468 "GET /admin/dz_karmaapp/product/ HTTP/1.1" 200 10514
INFO 2023-09-23 11:41:15,137 basehttp 14184 16468 "GET /admin/jsi18n/ HTTP/1.1" 200 16066
INFO 2023-09-23 11:41:18,215 basehttp 14184 16468 "GET /admin/dz_karmaapp/product/ HTTP/1.1" 200 10259
INFO 2023-09-23 11:41:18,229 basehttp 14184 16468 "GET /admin/jsi18n/ HTTP/1.1" 200 16066
INFO 2023-09-23 11:41:21,931 basehttp 14184 16468 "GET /admin/dz_karmaapp/category/ HTTP/1.1" 200 9899
INFO 2023-09-23 11:41:21,945 basehttp 14184 16468 "GET /admin/jsi18n/ HTTP/1.1" 200 16066
INFO 2023-09-23 11:41:23,812 basehttp 14184 16468 "GET /admin/auth/group/ HTTP/1.1" 200 8538
INFO 2023-09-23 11:41:23,827 basehttp 14184 16468 "GET /admin/jsi18n/ HTTP/1.1" 200 16066
INFO 2023-09-23 11:41:25,190 basehttp 14184 16468 "GET /admin/auth/user/ HTTP/1.1" 200 12585
INFO 2023-09-23 11:41:25,203 basehttp 14184 10060 "GET /static/admin/img/icon-no.svg HTTP/1.1" 200 560
INFO 2023-09-23 11:41:25,205 basehttp 14184 16468 "GET /admin/jsi18n/ HTTP/1.1" 200 16066
INFO 2023-09-23 11:41:29,402 basehttp 14184 16468 "GET /admin/auth/group/add/ HTTP/1.1" 200 13319
INFO 2023-09-23 11:41:29,416 basehttp 14184 16468 "GET /admin/jsi18n/ HTTP/1.1" 200 16066
INFO 2023-09-23 11:42:05,424 basehttp 14184 16468 "POST /admin/auth/group/add/ HTTP/1.1" 302 0
INFO 2023-09-23 11:42:05,435 basehttp 14184 16468 "GET /admin/auth/group/ HTTP/1.1" 200 10007
INFO 2023-09-23 11:42:05,448 basehttp 14184 16468 "GET /admin/jsi18n/ HTTP/1.1" 200 16066
INFO 2023-09-23 11:42:11,114 basehttp 14184 16468 "GET /admin/auth/group/1/change/ HTTP/1.1" 200 13559
INFO 2023-09-23 11:42:11,262 basehttp 14184 16468 "GET /admin/jsi18n/ HTTP/1.1" 200 16066
INFO 2023-09-23 11:42:16,551 basehttp 14184 16468 "POST /admin/auth/group/1/change/ HTTP/1.1" 302 0
INFO 2023-09-23 11:42:16,563 basehttp 14184 16468 "GET /admin/auth/group/ HTTP/1.1" 200 10007
INFO 2023-09-23 11:42:16,578 basehttp 14184 16468 "GET /admin/jsi18n/ HTTP/1.1" 200 16066
INFO 2023-09-23 11:42:19,620 basehttp 14184 16468 "GET /admin/auth/user/ HTTP/1.1" 200 12888
INFO 2023-09-23 11:42:19,634 basehttp 14184 16468 "GET /admin/jsi18n/ HTTP/1.1" 200 16066
INFO 2023-09-23 11:42:20,606 basehttp 14184 16468 "GET /admin/auth/user/2/change/ HTTP/1.1" 200 23671
INFO 2023-09-23 11:42:20,621 basehttp 14184 16468 "GET /admin/jsi18n/ HTTP/1.1" 200 16066
INFO 2023-09-23 11:42:30,392 basehttp 14184 16468 "POST /admin/auth/user/2/change/ HTTP/1.1" 302 0
INFO 2023-09-23 11:42:30,405 basehttp 14184 16468 "GET /admin/auth/user/ HTTP/1.1" 200 13109
INFO 2023-09-23 11:42:30,419 basehttp 14184 16468 "GET /admin/jsi18n/ HTTP/1.1" 200 16066
INFO 2023-09-23 11:45:32,835 autoreload 14184 20332 C:\Users\karmu\Desktop\STUDIUM\5 - Python\karmaproject\dz_karmaapp\models.py changed, reloading.
INFO 2023-09-23 11:45:33,271 autoreload 10092 6372 Watching for file changes with StatReloader
INFO 2023-09-23 11:46:03,173 autoreload 10092 6372 C:\Users\karmu\Desktop\STUDIUM\5 - Python\karmaproject\dz_karmaapp\models.py changed, reloading.
INFO 2023-09-23 11:46:03,604 autoreload 4736 23532 Watching for file changes with StatReloader
INFO 2023-09-23 11:46:15,027 autoreload 4736 23532 C:\Users\karmu\Desktop\STUDIUM\5 - Python\karmaproject\dz_karmaapp\models.py changed, reloading.
INFO 2023-09-23 11:46:15,384 autoreload 22964 22228 Watching for file changes with StatReloader
INFO 2023-09-23 11:49:36,588 autoreload 22964 22228 C:\Users\karmu\Desktop\STUDIUM\5 - Python\karmaproject\dz_karmaapp\models.py changed, reloading.
INFO 2023-09-23 11:49:36,959 autoreload 2548 19892 Watching for file changes with StatReloader
INFO 2023-09-23 11:50:35,512 autoreload 2548 19892 C:\Users\karmu\Desktop\STUDIUM\5 - Python\karmaproject\dz_karmaapp\models.py changed, reloading.
INFO 2023-09-23 11:50:35,871 autoreload 15220 22388 Watching for file changes with StatReloader
INFO 2023-09-23 13:24:03,862 autoreload 5436 20296 Watching for file changes with StatReloader
INFO 2023-09-23 13:24:07,366 basehttp 5436 23420 "GET /admin/dz_karmaapp/product/ HTTP/1.1" 200 10259
INFO 2023-09-23 13:24:07,385 basehttp 5436 23420 "GET /admin/jsi18n/ HTTP/1.1" 200 16066
INFO 2023-09-23 13:24:54,286 autoreload 5436 20296 C:\Users\karmu\Desktop\STUDIUM\5 - Python\karmaproject\dz_karmaapp\models.py changed, reloading.
INFO 2023-09-23 13:24:54,669 autoreload 16468 21296 Watching for file changes with StatReloader
INFO 2023-09-23 13:24:56,606 basehttp 16468 13780 "GET /admin/dz_karmaapp/product/ HTTP/1.1" 200 10091
INFO 2023-09-23 13:24:56,620 basehttp 16468 13780 "GET /admin/jsi18n/ HTTP/1.1" 200 16066
INFO 2023-09-23 13:25:34,858 autoreload 16468 21296 C:\Users\karmu\Desktop\STUDIUM\5 - Python\karmaproject\dz_karmaapp\admin.py changed, reloading.
INFO 2023-09-23 13:25:35,217 autoreload 13844 2596 Watching for file changes with StatReloader
INFO 2023-09-23 13:25:40,539 autoreload 13844 2596 C:\Users\karmu\Desktop\STUDIUM\5 - Python\karmaproject\dz_karmaapp\admin.py changed, reloading.
INFO 2023-09-23 13:25:40,896 autoreload 21596 6308 Watching for file changes with StatReloader
INFO 2023-09-23 13:25:47,235 autoreload 21596 6308 C:\Users\karmu\Desktop\STUDIUM\5 - Python\karmaproject\dz_karmaapp\admin.py changed, reloading.
INFO 2023-09-23 13:25:47,592 autoreload 15296 14624 Watching for file changes with StatReloader
INFO 2023-09-23 13:25:49,621 basehttp 15296 21888 "GET /admin/dz_karmaapp/product/ HTTP/1.1" 200 10091
INFO 2023-09-23 13:25:49,637 basehttp 15296 21888 "GET /admin/jsi18n/ HTTP/1.1" 200 16066
INFO 2023-09-23 13:25:52,819 basehttp 15296 21888 "GET /admin/dz_karmaapp/category/ HTTP/1.1" 200 9899
INFO 2023-09-23 13:25:52,834 basehttp 15296 21888 "GET /admin/jsi18n/ HTTP/1.1" 200 16066
INFO 2023-09-23 13:25:54,133 basehttp 15296 21888 "GET /admin/dz_karmaapp/product/ HTTP/1.1" 200 10091
INFO 2023-09-23 13:25:54,146 basehttp 15296 21888 "GET /admin/jsi18n/ HTTP/1.1" 200 16066
INFO 2023-09-23 13:25:59,221 basehttp 15296 21888 "GET /admin/auth/user/ HTTP/1.1" 200 12888
INFO 2023-09-23 13:25:59,234 basehttp 15296 21888 "GET /admin/jsi18n/ HTTP/1.1" 200 16066
INFO 2023-09-23 13:26:01,839 basehttp 15296 21888 "GET /admin/dz_karmaapp/product/ HTTP/1.1" 200 10091
INFO 2023-09-23 13:26:01,853 basehttp 15296 21888 "GET /admin/jsi18n/ HTTP/1.1" 200 16066
INFO 2023-09-23 13:27:23,176 autoreload 15296 14624 C:\Users\karmu\Desktop\STUDIUM\5 - Python\karmaproject\dz_karmaapp\admin.py changed, reloading.
INFO 2023-09-23 13:27:23,520 autoreload 21604 11964 Watching for file changes with StatReloader
INFO 2023-09-23 13:27:44,178 autoreload 21604 11964 C:\Users\karmu\Desktop\STUDIUM\5 - Python\karmaproject\dz_karmaapp\admin.py changed, reloading.
INFO 2023-09-23 13:27:44,536 autoreload 17712 22880 Watching for file changes with StatReloader
INFO 2023-09-23 13:27:57,049 basehttp 17712 15104 "GET /admin/dz_karmaapp/product/ HTTP/1.1" 200 10091
INFO 2023-09-23 13:27:57,064 basehttp 17712 15104 "GET /admin/jsi18n/ HTTP/1.1" 200 16066
INFO 2023-09-23 13:27:58,582 basehttp 17712 15104 "GET /admin/dz_karmaapp/product/ HTTP/1.1" 200 10091
INFO 2023-09-23 13:27:58,595 basehttp 17712 15104 "GET /admin/jsi18n/ HTTP/1.1" 200 16066
INFO 2023-09-23 13:28:39,482 autoreload 10700 1908 Watching for file changes with StatReloader
INFO 2023-09-23 13:28:43,719 basehttp 10700 2320 "GET /admin/dz_karmaapp/product/ HTTP/1.1" 200 10091
INFO 2023-09-23 13:28:43,735 basehttp 10700 2320 "GET /admin/jsi18n/ HTTP/1.1" 200 16066
INFO 2023-09-23 13:28:45,721 basehttp 10700 2320 "GET /admin/dz_karmaapp/product/4/change/ HTTP/1.1" 200 13995
INFO 2023-09-23 13:28:45,748 basehttp 10700 2320 "GET /admin/jsi18n/ HTTP/1.1" 200 16066
INFO 2023-09-23 13:28:53,403 basehttp 10700 2320 "GET /admin/dz_karmaapp/product/ HTTP/1.1" 200 10091
INFO 2023-09-23 13:28:53,418 basehttp 10700 2320 "GET /admin/jsi18n/ HTTP/1.1" 200 16066
INFO 2023-09-23 15:55:58,050 basehttp 10700 8724 "GET /admin/dz_karmaapp/product/ HTTP/1.1" 200 10091
INFO 2023-09-23 15:55:58,082 basehttp 10700 8724 "GET /admin/jsi18n/ HTTP/1.1" 200 16066
INFO 2023-09-23 18:14:34,780 basehttp 10700 8376 "GET /admin/dz_karmaapp/product/ HTTP/1.1" 200 10091
INFO 2023-09-23 18:14:34,800 basehttp 10700 8376 "GET /admin/jsi18n/ HTTP/1.1" 200 16066
INFO 2023-09-24 10:16:26,857 basehttp 10700 22556 "GET /admin/dz_karmaapp/product/ HTTP/1.1" 200 10091
INFO 2023-09-24 10:16:26,873 basehttp 10700 22556 "GET /admin/jsi18n/ HTTP/1.1" 200 16066
INFO 2023-09-24 10:16:33,066 basehttp 10700 22556 "GET /admin/dz_karmaapp/product/add/ HTTP/1.1" 200 13464
INFO 2023-09-24 10:16:33,079 basehttp 10700 22556 "GET /admin/jsi18n/ HTTP/1.1" 200 16066
INFO 2023-09-24 10:16:39,111 basehttp 10700 22556 "GET /admin/dz_karmaapp/product/ HTTP/1.1" 200 10091
INFO 2023-09-24 10:16:39,128 basehttp 10700 22556 "GET /admin/jsi18n/ HTTP/1.1" 200 16066
INFO 2023-09-24 10:16:40,799 basehttp 10700 22556 "GET /admin/dz_karmaapp/category/ HTTP/1.1" 200 9899
INFO 2023-09-24 10:16:40,812 basehttp 10700 22556 "GET /admin/jsi18n/ HTTP/1.1" 200 16066
INFO 2023-09-24 10:16:42,122 basehttp 10700 22556 "GET /admin/dz_karmaapp/product/ HTTP/1.1" 200 10091
INFO 2023-09-24 10:16:42,136 basehttp 10700 22556 "GET /admin/jsi18n/ HTTP/1.1" 200 16066
INFO 2023-09-24 10:16:44,776 basehttp 10700 22556 "GET /admin/auth/user/ HTTP/1.1" 200 12888
INFO 2023-09-24 10:16:44,790 basehttp 10700 22556 "GET /admin/jsi18n/ HTTP/1.1" 200 16066
INFO 2023-09-24 10:16:49,408 basehttp 10700 22556 "GET /admin/auth/group/ HTTP/1.1" 200 9795
INFO 2023-09-24 10:16:49,423 basehttp 10700 22556 "GET /admin/jsi18n/ HTTP/1.1" 200 16066
INFO 2023-09-24 10:16:50,322 basehttp 10700 22556 "GET /admin/auth/user/ HTTP/1.1" 200 12888
INFO 2023-09-24 10:16:50,336 basehttp 10700 22556 "GET /admin/jsi18n/ HTTP/1.1" 200 16066
INFO 2023-09-24 10:16:52,275 basehttp 10700 22556 "GET /admin/dz_karmaapp/product/ HTTP/1.1" 200 10091
INFO 2023-09-24 10:16:52,290 basehttp 10700 22556 "GET /admin/jsi18n/ HTTP/1.1" 200 16066
INFO 2023-09-24 10:16:53,182 basehttp 10700 22556 "GET /admin/auth/user/ HTTP/1.1" 200 12888
INFO 2023-09-24 10:16:53,195 basehttp 10700 22556 "GET /admin/jsi18n/ HTTP/1.1" 200 16066
INFO 2023-09-24 10:21:29,226 autoreload 10700 1908 C:\Users\karmu\Desktop\STUDIUM\5 - Python\karmaproject\dz_karmaapp\admin.py changed, reloading.
INFO 2023-09-24 10:21:29,797 autoreload 26208 18168 Watching for file changes with StatReloader
INFO 2023-09-24 10:21:33,836 basehttp 26208 21140 "GET /admin/auth/user/ HTTP/1.1" 200 12888
INFO 2023-09-24 10:21:33,852 basehttp 26208 21140 "GET /admin/jsi18n/ HTTP/1.1" 200 16066
INFO 2023-09-24 10:21:35,634 basehttp 26208 21140 "GET /admin/dz_karmaapp/product/ HTTP/1.1" 200 13442
INFO 2023-09-24 10:21:35,648 basehttp 26208 21140 "GET /admin/jsi18n/ HTTP/1.1" 200 16066
INFO 2023-09-24 10:21:52,531 basehttp 26208 21140 "POST /admin/dz_karmaapp/product/ HTTP/1.1" 302 0
INFO 2023-09-24 10:21:52,544 basehttp 26208 21140 "GET /admin/dz_karmaapp/product/ HTTP/1.1" 200 13442
INFO 2023-09-24 10:21:52,558 basehttp 26208 21140 "GET /admin/jsi18n/ HTTP/1.1" 200 16066
INFO 2023-09-24 10:26:22,272 autoreload 26208 18168 C:\Users\karmu\Desktop\STUDIUM\5 - Python\karmaproject\dz_karmaapp\admin.py changed, reloading.
INFO 2023-09-24 10:26:22,666 autoreload 21060 25932 Watching for file changes with StatReloader
INFO 2023-09-24 10:26:54,538 autoreload 21060 25932 C:\Users\karmu\Desktop\STUDIUM\5 - Python\karmaproject\dz_karmaapp\admin.py changed, reloading.
INFO 2023-09-24 10:26:54,883 autoreload 25604 16356 Watching for file changes with StatReloader
INFO 2023-09-24 10:26:58,808 basehttp 25604 24712 "GET /admin/dz_karmaapp/product/ HTTP/1.1" 200 13442
INFO 2023-09-24 10:26:58,823 basehttp 25604 24712 "GET /admin/jsi18n/ HTTP/1.1" 200 16066
INFO 2023-09-24 10:27:00,897 basehttp 25604 24712 "GET /admin/dz_karmaapp/product/2/change/ HTTP/1.1" 200 12398
INFO 2023-09-24 10:27:00,911 basehttp 25604 24712 "GET /admin/jsi18n/ HTTP/1.1" 200 16066
INFO 2023-09-24 10:32:45,083 autoreload 25604 16356 C:\Users\karmu\Desktop\STUDIUM\5 - Python\karmaproject\dz_karmaapp\admin.py changed, reloading.
INFO 2023-09-24 10:32:45,433 autoreload 11252 6360 Watching for file changes with StatReloader
INFO 2023-09-24 10:34:10,172 autoreload 11252 6360 C:\Users\karmu\Desktop\STUDIUM\5 - Python\karmaproject\dz_karmaapp\admin.py changed, reloading.
INFO 2023-09-24 10:34:10,540 autoreload 20848 19756 Watching for file changes with StatReloader
INFO 2023-09-24 10:35:07,753 autoreload 20848 19756 C:\Users\karmu\Desktop\STUDIUM\5 - Python\karmaproject\dz_karmaapp\admin.py changed, reloading.
INFO 2023-09-24 10:35:08,103 autoreload 16964 18412 Watching for file changes with StatReloader
INFO 2023-09-24 10:36:07,696 autoreload 16964 18412 C:\Users\karmu\Desktop\STUDIUM\5 - Python\karmaproject\dz_karmaapp\models.py changed, reloading.
INFO 2023-09-24 10:36:08,147 autoreload 25780 12196 Watching for file changes with StatReloader
INFO 2023-09-24 10:36:14,493 autoreload 25780 12196 C:\Users\karmu\Desktop\STUDIUM\5 - Python\karmaproject\dz_karmaapp\admin.py changed, reloading.
INFO 2023-09-24 10:36:14,868 autoreload 11220 20884 Watching for file changes with StatReloader
INFO 2023-09-24 10:37:09,365 autoreload 11220 20884 C:\Users\karmu\Desktop\STUDIUM\5 - Python\karmaproject\dz_karmaapp\models.py changed, reloading.
INFO 2023-09-24 10:37:09,725 autoreload 24100 19856 Watching for file changes with StatReloader
ERROR 2023-09-24 10:38:55,475 log 24100 4380 Internal Server Error: /admin/dz_karmaapp/product/2/change/
Traceback (most recent call last):
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\db\backends\utils.py", line 89, in _execute
    return self.cursor.execute(sql, params)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\db\backends\sqlite3\base.py", line 328, in execute
    return super().execute(query, params)
sqlite3.OperationalError: no such column: dz_karmaapp_product.rating

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\contrib\admin\options.py", line 688, in wrapper
    return self.admin_site.admin_view(view)(*args, **kwargs)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\utils\decorators.py", line 134, in _wrapper_view
    response = view_func(request, *args, **kwargs)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\views\decorators\cache.py", line 62, in _wrapper_view_func
    response = view_func(request, *args, **kwargs)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\contrib\admin\sites.py", line 242, in inner
    return view(request, *args, **kwargs)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\contrib\admin\options.py", line 1889, in change_view
    return self.changeform_view(request, object_id, form_url, extra_context)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\utils\decorators.py", line 46, in _wrapper
    return bound_method(*args, **kwargs)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\utils\decorators.py", line 134, in _wrapper_view
    response = view_func(request, *args, **kwargs)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\contrib\admin\options.py", line 1747, in changeform_view
    return self._changeform_view(request, object_id, form_url, extra_context)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\contrib\admin\options.py", line 1767, in _changeform_view
    obj = self.get_object(request, unquote(object_id), to_field)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\contrib\admin\options.py", line 866, in get_object
    return queryset.get(**{field.name: object_id})
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\db\models\query.py", line 633, in get
    num = len(clone)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\db\models\query.py", line 380, in __len__
    self._fetch_all()
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\db\models\query.py", line 1881, in _fetch_all
    self._result_cache = list(self._iterable_class(self))
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\db\models\query.py", line 91, in __iter__
    results = compiler.execute_sql(
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\db\models\sql\compiler.py", line 1562, in execute_sql
    cursor.execute(sql, params)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\db\backends\utils.py", line 102, in execute
    return super().execute(sql, params)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\db\backends\utils.py", line 67, in execute
    return self._execute_with_wrappers(
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\db\backends\utils.py", line 80, in _execute_with_wrappers
    return executor(sql, params, many, context)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\db\backends\utils.py", line 84, in _execute
    with self.db.wrap_database_errors:
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\db\utils.py", line 91, in __exit__
    raise dj_exc_value.with_traceback(traceback) from exc_value
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\db\backends\utils.py", line 89, in _execute
    return self.cursor.execute(sql, params)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\db\backends\sqlite3\base.py", line 328, in execute
    return super().execute(query, params)
django.db.utils.OperationalError: no such column: dz_karmaapp_product.rating
ERROR 2023-09-24 10:38:55,480 basehttp 24100 4380 "GET /admin/dz_karmaapp/product/2/change/ HTTP/1.1" 500 184142
INFO 2023-09-24 10:39:36,115 autoreload 24100 19856 C:\Users\karmu\Desktop\STUDIUM\5 - Python\karmaproject\dz_karmaapp\admin.py changed, reloading.
INFO 2023-09-24 10:39:36,490 autoreload 11596 22988 Watching for file changes with StatReloader
INFO 2023-09-24 10:39:44,877 autoreload 11596 22988 C:\Users\karmu\Desktop\STUDIUM\5 - Python\karmaproject\dz_karmaapp\admin.py changed, reloading.
INFO 2023-09-24 10:39:45,219 autoreload 18896 348 Watching for file changes with StatReloader
INFO 2023-09-24 10:40:01,752 autoreload 18896 348 C:\Users\karmu\Desktop\STUDIUM\5 - Python\karmaproject\dz_karmaapp\admin.py changed, reloading.
INFO 2023-09-24 10:40:02,118 autoreload 25252 25760 Watching for file changes with StatReloader
ERROR 2023-09-24 10:40:21,774 log 25252 19692 Internal Server Error: /admin/dz_karmaapp/product/2/change/
Traceback (most recent call last):
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\db\backends\utils.py", line 89, in _execute
    return self.cursor.execute(sql, params)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\db\backends\sqlite3\base.py", line 328, in execute
    return super().execute(query, params)
sqlite3.OperationalError: no such column: dz_karmaapp_product.rating

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\contrib\admin\options.py", line 688, in wrapper
    return self.admin_site.admin_view(view)(*args, **kwargs)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\utils\decorators.py", line 134, in _wrapper_view
    response = view_func(request, *args, **kwargs)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\views\decorators\cache.py", line 62, in _wrapper_view_func
    response = view_func(request, *args, **kwargs)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\contrib\admin\sites.py", line 242, in inner
    return view(request, *args, **kwargs)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\contrib\admin\options.py", line 1889, in change_view
    return self.changeform_view(request, object_id, form_url, extra_context)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\utils\decorators.py", line 46, in _wrapper
    return bound_method(*args, **kwargs)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\utils\decorators.py", line 134, in _wrapper_view
    response = view_func(request, *args, **kwargs)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\contrib\admin\options.py", line 1747, in changeform_view
    return self._changeform_view(request, object_id, form_url, extra_context)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\contrib\admin\options.py", line 1767, in _changeform_view
    obj = self.get_object(request, unquote(object_id), to_field)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\contrib\admin\options.py", line 866, in get_object
    return queryset.get(**{field.name: object_id})
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\db\models\query.py", line 633, in get
    num = len(clone)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\db\models\query.py", line 380, in __len__
    self._fetch_all()
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\db\models\query.py", line 1881, in _fetch_all
    self._result_cache = list(self._iterable_class(self))
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\db\models\query.py", line 91, in __iter__
    results = compiler.execute_sql(
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\db\models\sql\compiler.py", line 1562, in execute_sql
    cursor.execute(sql, params)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\db\backends\utils.py", line 102, in execute
    return super().execute(sql, params)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\db\backends\utils.py", line 67, in execute
    return self._execute_with_wrappers(
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\db\backends\utils.py", line 80, in _execute_with_wrappers
    return executor(sql, params, many, context)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\db\backends\utils.py", line 84, in _execute
    with self.db.wrap_database_errors:
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\db\utils.py", line 91, in __exit__
    raise dj_exc_value.with_traceback(traceback) from exc_value
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\db\backends\utils.py", line 89, in _execute
    return self.cursor.execute(sql, params)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\db\backends\sqlite3\base.py", line 328, in execute
    return super().execute(query, params)
django.db.utils.OperationalError: no such column: dz_karmaapp_product.rating
ERROR 2023-09-24 10:40:21,780 basehttp 25252 19692 "GET /admin/dz_karmaapp/product/2/change/ HTTP/1.1" 500 184142
INFO 2023-09-24 10:40:52,583 autoreload 25252 25760 C:\Users\karmu\Desktop\STUDIUM\5 - Python\karmaproject\dz_karmaapp\forms.py changed, reloading.
INFO 2023-09-24 10:40:52,923 autoreload 18468 26104 Watching for file changes with StatReloader
ERROR 2023-09-24 10:40:55,580 log 18468 8156 Internal Server Error: /admin/dz_karmaapp/product/2/change/
Traceback (most recent call last):
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\db\backends\utils.py", line 89, in _execute
    return self.cursor.execute(sql, params)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\db\backends\sqlite3\base.py", line 328, in execute
    return super().execute(query, params)
sqlite3.OperationalError: no such column: dz_karmaapp_product.rating

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\contrib\admin\options.py", line 688, in wrapper
    return self.admin_site.admin_view(view)(*args, **kwargs)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\utils\decorators.py", line 134, in _wrapper_view
    response = view_func(request, *args, **kwargs)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\views\decorators\cache.py", line 62, in _wrapper_view_func
    response = view_func(request, *args, **kwargs)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\contrib\admin\sites.py", line 242, in inner
    return view(request, *args, **kwargs)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\contrib\admin\options.py", line 1889, in change_view
    return self.changeform_view(request, object_id, form_url, extra_context)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\utils\decorators.py", line 46, in _wrapper
    return bound_method(*args, **kwargs)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\utils\decorators.py", line 134, in _wrapper_view
    response = view_func(request, *args, **kwargs)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\contrib\admin\options.py", line 1747, in changeform_view
    return self._changeform_view(request, object_id, form_url, extra_context)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\contrib\admin\options.py", line 1767, in _changeform_view
    obj = self.get_object(request, unquote(object_id), to_field)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\contrib\admin\options.py", line 866, in get_object
    return queryset.get(**{field.name: object_id})
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\db\models\query.py", line 633, in get
    num = len(clone)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\db\models\query.py", line 380, in __len__
    self._fetch_all()
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\db\models\query.py", line 1881, in _fetch_all
    self._result_cache = list(self._iterable_class(self))
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\db\models\query.py", line 91, in __iter__
    results = compiler.execute_sql(
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\db\models\sql\compiler.py", line 1562, in execute_sql
    cursor.execute(sql, params)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\db\backends\utils.py", line 102, in execute
    return super().execute(sql, params)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\db\backends\utils.py", line 67, in execute
    return self._execute_with_wrappers(
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\db\backends\utils.py", line 80, in _execute_with_wrappers
    return executor(sql, params, many, context)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\db\backends\utils.py", line 84, in _execute
    with self.db.wrap_database_errors:
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\db\utils.py", line 91, in __exit__
    raise dj_exc_value.with_traceback(traceback) from exc_value
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\db\backends\utils.py", line 89, in _execute
    return self.cursor.execute(sql, params)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\db\backends\sqlite3\base.py", line 328, in execute
    return super().execute(query, params)
django.db.utils.OperationalError: no such column: dz_karmaapp_product.rating
ERROR 2023-09-24 10:40:55,584 basehttp 18468 8156 "GET /admin/dz_karmaapp/product/2/change/ HTTP/1.1" 500 184142
ERROR 2023-09-24 10:40:56,506 log 18468 8156 Internal Server Error: /admin/dz_karmaapp/product/2/change/
Traceback (most recent call last):
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\db\backends\utils.py", line 89, in _execute
    return self.cursor.execute(sql, params)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\db\backends\sqlite3\base.py", line 328, in execute
    return super().execute(query, params)
sqlite3.OperationalError: no such column: dz_karmaapp_product.rating

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\contrib\admin\options.py", line 688, in wrapper
    return self.admin_site.admin_view(view)(*args, **kwargs)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\utils\decorators.py", line 134, in _wrapper_view
    response = view_func(request, *args, **kwargs)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\views\decorators\cache.py", line 62, in _wrapper_view_func
    response = view_func(request, *args, **kwargs)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\contrib\admin\sites.py", line 242, in inner
    return view(request, *args, **kwargs)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\contrib\admin\options.py", line 1889, in change_view
    return self.changeform_view(request, object_id, form_url, extra_context)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\utils\decorators.py", line 46, in _wrapper
    return bound_method(*args, **kwargs)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\utils\decorators.py", line 134, in _wrapper_view
    response = view_func(request, *args, **kwargs)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\contrib\admin\options.py", line 1747, in changeform_view
    return self._changeform_view(request, object_id, form_url, extra_context)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\contrib\admin\options.py", line 1767, in _changeform_view
    obj = self.get_object(request, unquote(object_id), to_field)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\contrib\admin\options.py", line 866, in get_object
    return queryset.get(**{field.name: object_id})
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\db\models\query.py", line 633, in get
    num = len(clone)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\db\models\query.py", line 380, in __len__
    self._fetch_all()
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\db\models\query.py", line 1881, in _fetch_all
    self._result_cache = list(self._iterable_class(self))
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\db\models\query.py", line 91, in __iter__
    results = compiler.execute_sql(
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\db\models\sql\compiler.py", line 1562, in execute_sql
    cursor.execute(sql, params)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\db\backends\utils.py", line 102, in execute
    return super().execute(sql, params)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\db\backends\utils.py", line 67, in execute
    return self._execute_with_wrappers(
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\db\backends\utils.py", line 80, in _execute_with_wrappers
    return executor(sql, params, many, context)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\db\backends\utils.py", line 84, in _execute
    with self.db.wrap_database_errors:
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\db\utils.py", line 91, in __exit__
    raise dj_exc_value.with_traceback(traceback) from exc_value
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\db\backends\utils.py", line 89, in _execute
    return self.cursor.execute(sql, params)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\db\backends\sqlite3\base.py", line 328, in execute
    return super().execute(query, params)
django.db.utils.OperationalError: no such column: dz_karmaapp_product.rating
ERROR 2023-09-24 10:40:56,510 basehttp 18468 8156 "GET /admin/dz_karmaapp/product/2/change/ HTTP/1.1" 500 184142
ERROR 2023-09-24 10:41:32,840 log 18468 8156 Internal Server Error: /admin/dz_karmaapp/product/
Traceback (most recent call last):
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\db\backends\utils.py", line 89, in _execute
    return self.cursor.execute(sql, params)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\db\backends\sqlite3\base.py", line 328, in execute
    return super().execute(query, params)
sqlite3.OperationalError: no such column: dz_karmaapp_product.rating

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\contrib\admin\options.py", line 688, in wrapper
    return self.admin_site.admin_view(view)(*args, **kwargs)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\utils\decorators.py", line 134, in _wrapper_view
    response = view_func(request, *args, **kwargs)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\views\decorators\cache.py", line 62, in _wrapper_view_func
    response = view_func(request, *args, **kwargs)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\contrib\admin\sites.py", line 242, in inner
    return view(request, *args, **kwargs)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\utils\decorators.py", line 46, in _wrapper
    return bound_method(*args, **kwargs)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\utils\decorators.py", line 134, in _wrapper_view
    response = view_func(request, *args, **kwargs)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\contrib\admin\options.py", line 2065, in changelist_view
    "selection_note": _("0 of %(cnt)s selected") % {"cnt": len(cl.result_list)},
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\db\models\query.py", line 380, in __len__
    self._fetch_all()
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\db\models\query.py", line 1881, in _fetch_all
    self._result_cache = list(self._iterable_class(self))
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\db\models\query.py", line 91, in __iter__
    results = compiler.execute_sql(
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\db\models\sql\compiler.py", line 1562, in execute_sql
    cursor.execute(sql, params)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\db\backends\utils.py", line 102, in execute
    return super().execute(sql, params)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\db\backends\utils.py", line 67, in execute
    return self._execute_with_wrappers(
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\db\backends\utils.py", line 80, in _execute_with_wrappers
    return executor(sql, params, many, context)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\db\backends\utils.py", line 84, in _execute
    with self.db.wrap_database_errors:
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\db\utils.py", line 91, in __exit__
    raise dj_exc_value.with_traceback(traceback) from exc_value
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\db\backends\utils.py", line 89, in _execute
    return self.cursor.execute(sql, params)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\db\backends\sqlite3\base.py", line 328, in execute
    return super().execute(query, params)
django.db.utils.OperationalError: no such column: dz_karmaapp_product.rating
ERROR 2023-09-24 10:41:32,845 basehttp 18468 8156 "GET /admin/dz_karmaapp/product/ HTTP/1.1" 500 170731
INFO 2023-09-24 10:41:35,010 basehttp 18468 8156 "GET /admin/dz_karmaapp/ HTTP/1.1" 200 5825
INFO 2023-09-24 10:42:59,467 autoreload 18628 12548 Watching for file changes with StatReloader
INFO 2023-09-24 10:43:02,094 basehttp 18628 20740 "GET /admin/dz_karmaapp/ HTTP/1.1" 200 5825
INFO 2023-09-24 10:43:03,335 basehttp 18628 20740 "GET /admin/dz_karmaapp/product/ HTTP/1.1" 200 13442
INFO 2023-09-24 10:43:03,350 basehttp 18628 20740 "GET /admin/jsi18n/ HTTP/1.1" 200 16066
INFO 2023-09-24 10:43:06,664 basehttp 18628 20740 "GET /admin/dz_karmaapp/product/1/change/ HTTP/1.1" 200 14922
INFO 2023-09-24 10:43:06,678 basehttp 18628 1500 "GET /admin/jsi18n/ HTTP/1.1" 200 16066
INFO 2023-09-24 10:43:06,715 basehttp 18628 20740 "GET /static/admin/js/collapse.js HTTP/1.1" 200 1803
INFO 2023-09-24 11:24:44,084 autoreload 19064 23716 Watching for file changes with StatReloader
INFO 2023-09-24 11:24:48,781 basehttp 19064 20748 "GET /admin/dz_karmaapp/product/1/change/ HTTP/1.1" 200 15152
INFO 2023-09-24 11:24:48,797 basehttp 19064 20748 "GET /admin/jsi18n/ HTTP/1.1" 200 16066
INFO 2023-09-24 11:24:51,258 basehttp 19064 20748 "GET /admin/dz_karmaapp/product/ HTTP/1.1" 200 13755
INFO 2023-09-24 11:24:51,271 basehttp 19064 20748 "GET /admin/jsi18n/ HTTP/1.1" 200 16066
INFO 2023-09-24 11:24:53,023 basehttp 19064 20748 "GET /admin/dz_karmaapp/client/ HTTP/1.1" 200 9848
INFO 2023-09-24 11:24:53,040 basehttp 19064 20748 "GET /admin/jsi18n/ HTTP/1.1" 200 16066
ERROR 2023-09-24 11:24:56,781 log 19064 20748 Internal Server Error: /admin/dz_karmaapp/client/add/
Traceback (most recent call last):
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\contrib\admin\options.py", line 688, in wrapper
    return self.admin_site.admin_view(view)(*args, **kwargs)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\utils\decorators.py", line 134, in _wrapper_view
    response = view_func(request, *args, **kwargs)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\views\decorators\cache.py", line 62, in _wrapper_view_func
    response = view_func(request, *args, **kwargs)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\contrib\admin\sites.py", line 242, in inner
    return view(request, *args, **kwargs)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\contrib\admin\options.py", line 1886, in add_view
    return self.changeform_view(request, None, form_url, extra_context)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\utils\decorators.py", line 46, in _wrapper
    return bound_method(*args, **kwargs)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\utils\decorators.py", line 134, in _wrapper_view
    response = view_func(request, *args, **kwargs)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\contrib\admin\options.py", line 1747, in changeform_view
    return self._changeform_view(request, object_id, form_url, extra_context)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\contrib\admin\options.py", line 1838, in _changeform_view
    media = self.media + admin_form.media
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\contrib\admin\helpers.py", line 97, in media
    for fs in self:
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\contrib\admin\helpers.py", line 70, in __iter__
    yield Fieldset(
TypeError: Fieldset.__init__() got an unexpected keyword argument 'email'
ERROR 2023-09-24 11:24:56,785 basehttp 19064 20748 "GET /admin/dz_karmaapp/client/add/ HTTP/1.1" 500 122133
WARNING 2023-09-24 11:28:39,409 log 19064 20748 Not Found: /client/add
WARNING 2023-09-24 11:28:39,410 basehttp 19064 20748 "GET /client/add HTTP/1.1" 404 2544
INFO 2023-09-24 11:28:57,118 basehttp 19064 20748 "GET /dz/client/add HTTP/1.1" 301 0
INFO 2023-09-24 11:28:57,127 basehttp 19064 13012 "GET /dz/client/add/ HTTP/1.1" 200 1245
INFO 2023-09-24 11:29:41,377 basehttp 19064 13012 "POST /dz/client/add/ HTTP/1.1" 200 1318
INFO 2023-09-24 11:29:53,487 basehttp 19064 13012 "POST /dz/client/add/ HTTP/1.1" 200 1333
ERROR 2023-09-24 11:30:03,957 log 19064 13012 Internal Server Error: /dz/client/add/
Traceback (most recent call last):
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\db\backends\utils.py", line 89, in _execute
    return self.cursor.execute(sql, params)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\db\backends\sqlite3\base.py", line 328, in execute
    return super().execute(query, params)
sqlite3.IntegrityError: UNIQUE constraint failed: dz_karmaapp_client.tel

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\karmaproject\dz_karmaapp\views.py", line 44, in client_form
    client.save()
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\db\models\base.py", line 814, in save
    self.save_base(
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\db\models\base.py", line 877, in save_base
    updated = self._save_table(
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\db\models\base.py", line 1020, in _save_table
    results = self._do_insert(
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\db\models\base.py", line 1061, in _do_insert
    return manager._insert(
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\db\models\manager.py", line 87, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\db\models\query.py", line 1805, in _insert
    return query.get_compiler(using=using).execute_sql(returning_fields)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\db\models\sql\compiler.py", line 1822, in execute_sql
    cursor.execute(sql, params)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\db\backends\utils.py", line 102, in execute
    return super().execute(sql, params)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\db\backends\utils.py", line 67, in execute
    return self._execute_with_wrappers(
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\db\backends\utils.py", line 80, in _execute_with_wrappers
    return executor(sql, params, many, context)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\db\backends\utils.py", line 84, in _execute
    with self.db.wrap_database_errors:
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\db\utils.py", line 91, in __exit__
    raise dj_exc_value.with_traceback(traceback) from exc_value
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\db\backends\utils.py", line 89, in _execute
    return self.cursor.execute(sql, params)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\db\backends\sqlite3\base.py", line 328, in execute
    return super().execute(query, params)
django.db.utils.IntegrityError: UNIQUE constraint failed: dz_karmaapp_client.tel
ERROR 2023-09-24 11:30:03,962 basehttp 19064 13012 "POST /dz/client/add/ HTTP/1.1" 500 155858
ERROR 2023-09-24 11:30:11,267 log 19064 13012 Internal Server Error: /dz/client/add/
Traceback (most recent call last):
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\db\backends\utils.py", line 89, in _execute
    return self.cursor.execute(sql, params)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\db\backends\sqlite3\base.py", line 328, in execute
    return super().execute(query, params)
sqlite3.IntegrityError: UNIQUE constraint failed: dz_karmaapp_client.tel

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\karmaproject\dz_karmaapp\views.py", line 44, in client_form
    client.save()
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\db\models\base.py", line 814, in save
    self.save_base(
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\db\models\base.py", line 877, in save_base
    updated = self._save_table(
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\db\models\base.py", line 1020, in _save_table
    results = self._do_insert(
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\db\models\base.py", line 1061, in _do_insert
    return manager._insert(
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\db\models\manager.py", line 87, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\db\models\query.py", line 1805, in _insert
    return query.get_compiler(using=using).execute_sql(returning_fields)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\db\models\sql\compiler.py", line 1822, in execute_sql
    cursor.execute(sql, params)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\db\backends\utils.py", line 102, in execute
    return super().execute(sql, params)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\db\backends\utils.py", line 67, in execute
    return self._execute_with_wrappers(
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\db\backends\utils.py", line 80, in _execute_with_wrappers
    return executor(sql, params, many, context)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\db\backends\utils.py", line 84, in _execute
    with self.db.wrap_database_errors:
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\db\utils.py", line 91, in __exit__
    raise dj_exc_value.with_traceback(traceback) from exc_value
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\db\backends\utils.py", line 89, in _execute
    return self.cursor.execute(sql, params)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\db\backends\sqlite3\base.py", line 328, in execute
    return super().execute(query, params)
django.db.utils.IntegrityError: UNIQUE constraint failed: dz_karmaapp_client.tel
ERROR 2023-09-24 11:30:11,270 basehttp 19064 13012 "POST /dz/client/add/ HTTP/1.1" 500 155869
INFO 2023-09-24 11:30:35,664 basehttp 19064 13012 "POST /dz/client/add/ HTTP/1.1" 200 1328
WARNING 2023-09-24 11:31:27,780 log 19064 13012 Not Found: /dz/admin
WARNING 2023-09-24 11:31:27,781 basehttp 19064 13012 "GET /dz/admin HTTP/1.1" 404 3341
INFO 2023-09-24 11:31:32,482 basehttp 19064 13012 "GET /admin HTTP/1.1" 301 0
INFO 2023-09-24 11:31:32,495 basehttp 19064 24120 "GET /admin/ HTTP/1.1" 200 10622
INFO 2023-09-24 11:31:40,475 basehttp 19064 24120 "GET /admin/dz_karmaapp/client/ HTTP/1.1" 200 12498
INFO 2023-09-24 11:31:40,491 basehttp 19064 24120 "GET /admin/jsi18n/ HTTP/1.1" 200 16066
INFO 2023-09-24 11:31:53,463 basehttp 19064 24120 "POST /admin/dz_karmaapp/client/ HTTP/1.1" 200 8687
INFO 2023-09-24 11:31:53,510 basehttp 19064 24120 "GET /static/admin/js/cancel.js HTTP/1.1" 200 884
INFO 2023-09-24 11:31:55,694 basehttp 19064 24120 "GET /admin/dz_karmaapp/client/ HTTP/1.1" 200 12498
ERROR 2023-09-24 11:31:58,770 log 19064 24120 Internal Server Error: /admin/dz_karmaapp/client/add/
Traceback (most recent call last):
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\contrib\admin\options.py", line 688, in wrapper
    return self.admin_site.admin_view(view)(*args, **kwargs)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\utils\decorators.py", line 134, in _wrapper_view
    response = view_func(request, *args, **kwargs)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\views\decorators\cache.py", line 62, in _wrapper_view_func
    response = view_func(request, *args, **kwargs)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\contrib\admin\sites.py", line 242, in inner
    return view(request, *args, **kwargs)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\contrib\admin\options.py", line 1886, in add_view
    return self.changeform_view(request, None, form_url, extra_context)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\utils\decorators.py", line 46, in _wrapper
    return bound_method(*args, **kwargs)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\utils\decorators.py", line 134, in _wrapper_view
    response = view_func(request, *args, **kwargs)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\contrib\admin\options.py", line 1747, in changeform_view
    return self._changeform_view(request, object_id, form_url, extra_context)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\contrib\admin\options.py", line 1838, in _changeform_view
    media = self.media + admin_form.media
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\contrib\admin\helpers.py", line 97, in media
    for fs in self:
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\contrib\admin\helpers.py", line 70, in __iter__
    yield Fieldset(
TypeError: Fieldset.__init__() got an unexpected keyword argument 'email'
ERROR 2023-09-24 11:31:58,773 basehttp 19064 24120 "GET /admin/dz_karmaapp/client/add/ HTTP/1.1" 500 122133
INFO 2023-09-24 11:32:00,975 basehttp 19064 24120 "GET /admin/dz_karmaapp/client/ HTTP/1.1" 200 12498
ERROR 2023-09-24 11:32:02,854 log 19064 24120 Internal Server Error: /admin/dz_karmaapp/client/add/
Traceback (most recent call last):
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\contrib\admin\options.py", line 688, in wrapper
    return self.admin_site.admin_view(view)(*args, **kwargs)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\utils\decorators.py", line 134, in _wrapper_view
    response = view_func(request, *args, **kwargs)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\views\decorators\cache.py", line 62, in _wrapper_view_func
    response = view_func(request, *args, **kwargs)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\contrib\admin\sites.py", line 242, in inner
    return view(request, *args, **kwargs)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\contrib\admin\options.py", line 1886, in add_view
    return self.changeform_view(request, None, form_url, extra_context)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\utils\decorators.py", line 46, in _wrapper
    return bound_method(*args, **kwargs)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\utils\decorators.py", line 134, in _wrapper_view
    response = view_func(request, *args, **kwargs)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\contrib\admin\options.py", line 1747, in changeform_view
    return self._changeform_view(request, object_id, form_url, extra_context)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\contrib\admin\options.py", line 1838, in _changeform_view
    media = self.media + admin_form.media
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\contrib\admin\helpers.py", line 97, in media
    for fs in self:
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\contrib\admin\helpers.py", line 70, in __iter__
    yield Fieldset(
TypeError: Fieldset.__init__() got an unexpected keyword argument 'email'
ERROR 2023-09-24 11:32:02,855 basehttp 19064 24120 "GET /admin/dz_karmaapp/client/add/ HTTP/1.1" 500 122133
INFO 2023-09-24 11:32:05,177 basehttp 19064 24120 "GET /admin/dz_karmaapp/client/ HTTP/1.1" 200 12498
INFO 2023-09-24 11:32:06,296 basehttp 19064 24120 "GET /admin/dz_karmaapp/product/add/ HTTP/1.1" 200 14899
INFO 2023-09-24 11:32:06,312 basehttp 19064 24120 "GET /admin/jsi18n/ HTTP/1.1" 200 16066
ERROR 2023-09-24 11:32:11,630 log 19064 24120 Internal Server Error: /admin/dz_karmaapp/client/add/
Traceback (most recent call last):
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\contrib\admin\options.py", line 688, in wrapper
    return self.admin_site.admin_view(view)(*args, **kwargs)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\utils\decorators.py", line 134, in _wrapper_view
    response = view_func(request, *args, **kwargs)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\views\decorators\cache.py", line 62, in _wrapper_view_func
    response = view_func(request, *args, **kwargs)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\contrib\admin\sites.py", line 242, in inner
    return view(request, *args, **kwargs)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\contrib\admin\options.py", line 1886, in add_view
    return self.changeform_view(request, None, form_url, extra_context)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\utils\decorators.py", line 46, in _wrapper
    return bound_method(*args, **kwargs)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\utils\decorators.py", line 134, in _wrapper_view
    response = view_func(request, *args, **kwargs)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\contrib\admin\options.py", line 1747, in changeform_view
    return self._changeform_view(request, object_id, form_url, extra_context)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\contrib\admin\options.py", line 1838, in _changeform_view
    media = self.media + admin_form.media
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\contrib\admin\helpers.py", line 97, in media
    for fs in self:
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\contrib\admin\helpers.py", line 70, in __iter__
    yield Fieldset(
TypeError: Fieldset.__init__() got an unexpected keyword argument 'email'
ERROR 2023-09-24 11:32:11,633 basehttp 19064 24120 "GET /admin/dz_karmaapp/client/add/ HTTP/1.1" 500 122138
INFO 2023-09-24 11:33:02,531 autoreload 19064 23716 C:\Users\karmu\Desktop\STUDIUM\5 - Python\karmaproject\dz_karmaapp\models.py changed, reloading.
INFO 2023-09-24 11:33:02,945 autoreload 25708 20004 Watching for file changes with StatReloader
ERROR 2023-09-24 11:33:06,837 log 25708 1348 Internal Server Error: /admin/dz_karmaapp/client/add/
Traceback (most recent call last):
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\contrib\admin\options.py", line 688, in wrapper
    return self.admin_site.admin_view(view)(*args, **kwargs)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\utils\decorators.py", line 134, in _wrapper_view
    response = view_func(request, *args, **kwargs)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\views\decorators\cache.py", line 62, in _wrapper_view_func
    response = view_func(request, *args, **kwargs)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\contrib\admin\sites.py", line 242, in inner
    return view(request, *args, **kwargs)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\contrib\admin\options.py", line 1886, in add_view
    return self.changeform_view(request, None, form_url, extra_context)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\utils\decorators.py", line 46, in _wrapper
    return bound_method(*args, **kwargs)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\utils\decorators.py", line 134, in _wrapper_view
    response = view_func(request, *args, **kwargs)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\contrib\admin\options.py", line 1747, in changeform_view
    return self._changeform_view(request, object_id, form_url, extra_context)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\contrib\admin\options.py", line 1838, in _changeform_view
    media = self.media + admin_form.media
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\contrib\admin\helpers.py", line 97, in media
    for fs in self:
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\contrib\admin\helpers.py", line 70, in __iter__
    yield Fieldset(
TypeError: Fieldset.__init__() got an unexpected keyword argument 'email'
ERROR 2023-09-24 11:33:06,841 basehttp 25708 1348 "GET /admin/dz_karmaapp/client/add/ HTTP/1.1" 500 122275
INFO 2023-09-24 11:33:19,611 autoreload 25708 20004 C:\Users\karmu\Desktop\STUDIUM\5 - Python\karmaproject\dz_karmaapp\models.py changed, reloading.
INFO 2023-09-24 11:33:19,992 autoreload 15008 20732 Watching for file changes with StatReloader
ERROR 2023-09-24 11:33:22,135 log 15008 23780 Internal Server Error: /admin/dz_karmaapp/client/add/
Traceback (most recent call last):
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\contrib\admin\options.py", line 688, in wrapper
    return self.admin_site.admin_view(view)(*args, **kwargs)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\utils\decorators.py", line 134, in _wrapper_view
    response = view_func(request, *args, **kwargs)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\views\decorators\cache.py", line 62, in _wrapper_view_func
    response = view_func(request, *args, **kwargs)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\contrib\admin\sites.py", line 242, in inner
    return view(request, *args, **kwargs)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\contrib\admin\options.py", line 1886, in add_view
    return self.changeform_view(request, None, form_url, extra_context)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\utils\decorators.py", line 46, in _wrapper
    return bound_method(*args, **kwargs)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\utils\decorators.py", line 134, in _wrapper_view
    response = view_func(request, *args, **kwargs)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\contrib\admin\options.py", line 1747, in changeform_view
    return self._changeform_view(request, object_id, form_url, extra_context)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\contrib\admin\options.py", line 1838, in _changeform_view
    media = self.media + admin_form.media
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\contrib\admin\helpers.py", line 97, in media
    for fs in self:
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\contrib\admin\helpers.py", line 70, in __iter__
    yield Fieldset(
TypeError: Fieldset.__init__() got an unexpected keyword argument 'email'
ERROR 2023-09-24 11:33:22,138 basehttp 15008 23780 "GET /admin/dz_karmaapp/client/add/ HTTP/1.1" 500 122275
INFO 2023-09-24 11:33:45,902 autoreload 15008 20732 C:\Users\karmu\Desktop\STUDIUM\5 - Python\karmaproject\dz_karmaapp\models.py changed, reloading.
INFO 2023-09-24 11:33:46,266 autoreload 23636 6664 Watching for file changes with StatReloader
INFO 2023-09-24 11:34:32,560 autoreload 23636 6664 C:\Users\karmu\Desktop\STUDIUM\5 - Python\karmaproject\dz_karmaapp\models.py changed, reloading.
INFO 2023-09-24 11:34:32,908 autoreload 21792 26448 Watching for file changes with StatReloader
INFO 2023-09-24 11:34:52,173 basehttp 21792 10684 "GET /admin/dz_karmaapp/product/add/ HTTP/1.1" 200 14899
INFO 2023-09-24 11:34:54,526 basehttp 21792 10684 "GET /admin/dz_karmaapp/client/ HTTP/1.1" 200 12498
INFO 2023-09-24 11:34:54,540 basehttp 21792 10684 "GET /admin/jsi18n/ HTTP/1.1" 200 16066
ERROR 2023-09-24 11:35:33,190 log 21792 10684 Internal Server Error: /admin/dz_karmaapp/client/1/change/
Traceback (most recent call last):
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\contrib\admin\options.py", line 688, in wrapper
    return self.admin_site.admin_view(view)(*args, **kwargs)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\utils\decorators.py", line 134, in _wrapper_view
    response = view_func(request, *args, **kwargs)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\views\decorators\cache.py", line 62, in _wrapper_view_func
    response = view_func(request, *args, **kwargs)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\contrib\admin\sites.py", line 242, in inner
    return view(request, *args, **kwargs)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\contrib\admin\options.py", line 1889, in change_view
    return self.changeform_view(request, object_id, form_url, extra_context)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\utils\decorators.py", line 46, in _wrapper
    return bound_method(*args, **kwargs)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\utils\decorators.py", line 134, in _wrapper_view
    response = view_func(request, *args, **kwargs)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\contrib\admin\options.py", line 1747, in changeform_view
    return self._changeform_view(request, object_id, form_url, extra_context)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\contrib\admin\options.py", line 1838, in _changeform_view
    media = self.media + admin_form.media
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\contrib\admin\helpers.py", line 97, in media
    for fs in self:
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\contrib\admin\helpers.py", line 70, in __iter__
    yield Fieldset(
TypeError: Fieldset.__init__() got an unexpected keyword argument 'email'
ERROR 2023-09-24 11:35:33,193 basehttp 21792 10684 "GET /admin/dz_karmaapp/client/1/change/ HTTP/1.1" 500 122535
INFO 2023-09-24 11:35:34,726 basehttp 21792 10684 "GET /admin/dz_karmaapp/client/ HTTP/1.1" 200 12498
ERROR 2023-09-24 11:35:35,704 log 21792 10684 Internal Server Error: /admin/dz_karmaapp/client/1/change/
Traceback (most recent call last):
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\contrib\admin\options.py", line 688, in wrapper
    return self.admin_site.admin_view(view)(*args, **kwargs)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\utils\decorators.py", line 134, in _wrapper_view
    response = view_func(request, *args, **kwargs)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\views\decorators\cache.py", line 62, in _wrapper_view_func
    response = view_func(request, *args, **kwargs)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\contrib\admin\sites.py", line 242, in inner
    return view(request, *args, **kwargs)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\contrib\admin\options.py", line 1889, in change_view
    return self.changeform_view(request, object_id, form_url, extra_context)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\utils\decorators.py", line 46, in _wrapper
    return bound_method(*args, **kwargs)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\utils\decorators.py", line 134, in _wrapper_view
    response = view_func(request, *args, **kwargs)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\contrib\admin\options.py", line 1747, in changeform_view
    return self._changeform_view(request, object_id, form_url, extra_context)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\contrib\admin\options.py", line 1838, in _changeform_view
    media = self.media + admin_form.media
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\contrib\admin\helpers.py", line 97, in media
    for fs in self:
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\contrib\admin\helpers.py", line 70, in __iter__
    yield Fieldset(
TypeError: Fieldset.__init__() got an unexpected keyword argument 'email'
ERROR 2023-09-24 11:35:35,707 basehttp 21792 10684 "GET /admin/dz_karmaapp/client/1/change/ HTTP/1.1" 500 122535
INFO 2023-09-24 11:35:36,915 basehttp 21792 10684 "GET /admin/dz_karmaapp/client/ HTTP/1.1" 200 12498
ERROR 2023-09-24 11:35:37,869 log 21792 10684 Internal Server Error: /admin/dz_karmaapp/client/2/change/
Traceback (most recent call last):
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\contrib\admin\options.py", line 688, in wrapper
    return self.admin_site.admin_view(view)(*args, **kwargs)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\utils\decorators.py", line 134, in _wrapper_view
    response = view_func(request, *args, **kwargs)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\views\decorators\cache.py", line 62, in _wrapper_view_func
    response = view_func(request, *args, **kwargs)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\contrib\admin\sites.py", line 242, in inner
    return view(request, *args, **kwargs)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\contrib\admin\options.py", line 1889, in change_view
    return self.changeform_view(request, object_id, form_url, extra_context)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\utils\decorators.py", line 46, in _wrapper
    return bound_method(*args, **kwargs)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\utils\decorators.py", line 134, in _wrapper_view
    response = view_func(request, *args, **kwargs)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\contrib\admin\options.py", line 1747, in changeform_view
    return self._changeform_view(request, object_id, form_url, extra_context)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\contrib\admin\options.py", line 1838, in _changeform_view
    media = self.media + admin_form.media
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\contrib\admin\helpers.py", line 97, in media
    for fs in self:
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\contrib\admin\helpers.py", line 70, in __iter__
    yield Fieldset(
TypeError: Fieldset.__init__() got an unexpected keyword argument 'email'
ERROR 2023-09-24 11:35:37,871 basehttp 21792 10684 "GET /admin/dz_karmaapp/client/2/change/ HTTP/1.1" 500 122536
INFO 2023-09-24 11:35:39,107 basehttp 21792 10684 "GET /admin/dz_karmaapp/client/ HTTP/1.1" 200 12498
INFO 2023-09-24 11:36:20,665 autoreload 21792 26448 C:\Users\karmu\Desktop\STUDIUM\5 - Python\karmaproject\dz_karmaapp\admin.py changed, reloading.
INFO 2023-09-24 11:36:21,051 autoreload 18900 25560 Watching for file changes with StatReloader
ERROR 2023-09-24 11:36:22,733 log 18900 6208 Internal Server Error: /admin/dz_karmaapp/client/1/change/
Traceback (most recent call last):
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\contrib\admin\options.py", line 688, in wrapper
    return self.admin_site.admin_view(view)(*args, **kwargs)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\utils\decorators.py", line 134, in _wrapper_view
    response = view_func(request, *args, **kwargs)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\views\decorators\cache.py", line 62, in _wrapper_view_func
    response = view_func(request, *args, **kwargs)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\contrib\admin\sites.py", line 242, in inner
    return view(request, *args, **kwargs)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\contrib\admin\options.py", line 1889, in change_view
    return self.changeform_view(request, object_id, form_url, extra_context)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\utils\decorators.py", line 46, in _wrapper
    return bound_method(*args, **kwargs)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\utils\decorators.py", line 134, in _wrapper_view
    response = view_func(request, *args, **kwargs)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\contrib\admin\options.py", line 1747, in changeform_view
    return self._changeform_view(request, object_id, form_url, extra_context)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\contrib\admin\options.py", line 1838, in _changeform_view
    media = self.media + admin_form.media
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\contrib\admin\helpers.py", line 97, in media
    for fs in self:
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\contrib\admin\helpers.py", line 70, in __iter__
    yield Fieldset(
TypeError: Fieldset.__init__() got an unexpected keyword argument 'email'
ERROR 2023-09-24 11:36:22,735 basehttp 18900 6208 "GET /admin/dz_karmaapp/client/1/change/ HTTP/1.1" 500 122726
INFO 2023-09-24 11:36:54,079 autoreload 18900 25560 C:\Users\karmu\Desktop\STUDIUM\5 - Python\karmaproject\dz_karmaapp\admin.py changed, reloading.
INFO 2023-09-24 11:36:54,477 autoreload 26056 856 Watching for file changes with StatReloader
INFO 2023-09-24 11:36:55,820 basehttp 26056 25344 "GET /admin/dz_karmaapp/client/1/change/ HTTP/1.1" 200 12515
INFO 2023-09-24 11:36:55,836 basehttp 26056 25344 "GET /admin/jsi18n/ HTTP/1.1" 200 16066
INFO 2023-09-24 11:37:07,680 basehttp 26056 25344 "GET /admin/dz_karmaapp/client/add/ HTTP/1.1" 200 12214
INFO 2023-09-24 11:37:07,695 basehttp 26056 25344 "GET /admin/jsi18n/ HTTP/1.1" 200 16066
INFO 2023-09-24 11:37:27,073 basehttp 26056 25344 "POST /admin/dz_karmaapp/client/add/ HTTP/1.1" 302 0
INFO 2023-09-24 11:37:27,093 basehttp 26056 25344 "GET /admin/dz_karmaapp/client/ HTTP/1.1" 200 13041
INFO 2023-09-24 11:37:27,109 basehttp 26056 25344 "GET /admin/jsi18n/ HTTP/1.1" 200 16066
INFO 2023-09-24 11:37:29,162 basehttp 26056 25344 "GET /admin/dz_karmaapp/client/4/change/ HTTP/1.1" 200 12521
INFO 2023-09-24 11:37:29,178 basehttp 26056 25344 "GET /admin/jsi18n/ HTTP/1.1" 200 16066
INFO 2023-09-24 11:38:03,389 autoreload 26056 856 C:\Users\karmu\Desktop\STUDIUM\5 - Python\karmaproject\dz_karmaapp\admin.py changed, reloading.
INFO 2023-09-24 11:38:03,791 autoreload 16248 12556 Watching for file changes with StatReloader
INFO 2023-09-24 11:38:08,977 basehttp 16248 12896 "GET /admin/dz_karmaapp/client/4/change/ HTTP/1.1" 200 12527
INFO 2023-09-24 11:38:08,991 basehttp 16248 12896 "GET /admin/jsi18n/ HTTP/1.1" 200 16066
INFO 2023-09-24 11:53:13,571 autoreload 8580 23704 Watching for file changes with StatReloader
INFO 2023-09-24 11:53:19,618 basehttp 8580 8008 "GET / HTTP/1.1" 200 13
INFO 2023-09-24 11:53:33,937 basehttp 8580 8008 "GET /dz HTTP/1.1" 301 0
INFO 2023-09-24 11:53:33,951 basehttp 8580 21512 "GET /dz/ HTTP/1.1" 200 35
INFO 2023-09-24 11:54:09,877 basehttp 8580 21512 "GET /dz/ HTTP/1.1" 200 35
INFO 2023-09-24 11:54:11,777 basehttp 8580 21512 "GET /admin/dz_karmaapp/client/4/change/ HTTP/1.1" 200 22365
INFO 2023-09-24 11:54:11,793 basehttp 8580 21512 "GET /static/debug_toolbar/css/toolbar.css HTTP/1.1" 200 12621
INFO 2023-09-24 11:54:11,793 basehttp 8580 15392 "GET /static/debug_toolbar/js/toolbar.js HTTP/1.1" 200 13404
INFO 2023-09-24 11:54:11,806 basehttp 8580 21512 "GET /static/debug_toolbar/css/print.css HTTP/1.1" 200 43
INFO 2023-09-24 11:54:11,842 basehttp 8580 21512 "GET /static/debug_toolbar/js/utils.js HTTP/1.1" 200 4527
INFO 2023-09-24 11:54:13,408 basehttp 8580 21512 "GET /admin/dz_karmaapp/client/4/change/ HTTP/1.1" 200 22364
INFO 2023-09-24 11:54:13,437 basehttp 8580 21512 "GET /admin/jsi18n/ HTTP/1.1" 200 16066
INFO 2023-09-24 11:54:18,043 basehttp 8580 21512 "GET /admin/dz_karmaapp/ HTTP/1.1" 200 16095
INFO 2023-09-24 11:54:21,000 basehttp 8580 21512 "GET /admin HTTP/1.1" 301 0
INFO 2023-09-24 11:54:21,044 basehttp 8580 15392 "GET /admin/ HTTP/1.1" 200 20403
INFO 2023-09-24 11:54:23,666 basehttp 8580 15392 "GET / HTTP/1.1" 200 13
INFO 2023-09-24 11:54:33,991 basehttp 8580 15392 "GET /admin/ HTTP/1.1" 200 20403
INFO 2023-09-24 11:59:42,533 autoreload 8580 23704 C:\Users\karmu\Desktop\STUDIUM\5 - Python\karmaproject\dz_karmaapp\urls.py changed, reloading.
INFO 2023-09-24 11:59:42,966 autoreload 23896 5476 Watching for file changes with StatReloader
INFO 2023-09-24 12:02:06,909 autoreload 23896 5476 C:\Users\karmu\Desktop\STUDIUM\5 - Python\karmaproject\karmaproject\urls.py changed, reloading.
INFO 2023-09-24 12:02:07,311 autoreload 24252 14908 Watching for file changes with StatReloader
INFO 2023-09-24 12:02:16,822 autoreload 24252 14908 C:\Users\karmu\Desktop\STUDIUM\5 - Python\karmaproject\dz_karmaapp\urls.py changed, reloading.
INFO 2023-09-24 12:02:17,248 autoreload 15656 23616 Watching for file changes with StatReloader
INFO 2023-09-24 12:02:22,285 basehttp 15656 18736 "GET /admin/ HTTP/1.1" 200 20398
INFO 2023-09-24 12:02:22,297 basehttp 15656 5172 "GET /static/debug_toolbar/js/toolbar.js HTTP/1.1" 304 0
INFO 2023-09-24 12:02:22,297 basehttp 15656 18736 "GET /static/debug_toolbar/css/toolbar.css HTTP/1.1" 304 0
INFO 2023-09-24 12:02:22,316 basehttp 15656 18736 "GET /static/debug_toolbar/css/print.css HTTP/1.1" 304 0
INFO 2023-09-24 12:02:22,317 basehttp 15656 5172 "GET /static/debug_toolbar/js/utils.js HTTP/1.1" 304 0
INFO 2023-09-24 12:02:24,736 basehttp 15656 5172 "GET / HTTP/1.1" 200 13
INFO 2023-09-24 12:02:29,530 basehttp 15656 5172 "GET /dz/ HTTP/1.1" 200 35
INFO 2023-09-24 12:02:31,185 basehttp 15656 5172 "GET /dz/ HTTP/1.1" 200 35
INFO 2023-09-24 12:02:33,431 basehttp 15656 5172 "GET /admin/ HTTP/1.1" 200 20397
INFO 2023-09-24 12:02:45,854 autoreload 1636 21320 Watching for file changes with StatReloader
INFO 2023-09-24 12:02:49,005 basehttp 1636 22204 "GET /admin/ HTTP/1.1" 200 20398
INFO 2023-09-24 12:02:51,781 basehttp 1636 22204 "GET / HTTP/1.1" 200 13
INFO 2023-09-24 12:03:47,875 autoreload 20744 26216 Watching for file changes with StatReloader
INFO 2023-09-24 12:03:50,400 basehttp 20744 13712 "GET / HTTP/1.1" 200 13
INFO 2023-09-24 12:03:54,663 basehttp 20744 13712 "GET /about HTTP/1.1" 301 0
INFO 2023-09-24 12:03:54,676 basehttp 20744 15972 "GET /about/ HTTP/1.1" 200 27
INFO 2023-09-24 12:03:59,608 basehttp 20744 15972 "GET /dz/ HTTP/1.1" 200 35
INFO 2023-09-24 12:04:05,188 basehttp 20744 15972 "GET /dz/ HTTP/1.1" 200 35
INFO 2023-09-24 12:04:08,404 basehttp 20744 15972 "GET /dz/client/add HTTP/1.1" 301 0
INFO 2023-09-24 12:04:08,523 basehttp 20744 18464 "GET /dz/client/add/ HTTP/1.1" 200 11066
INFO 2023-09-24 12:04:08,535 basehttp 20744 18464 "GET /static/debug_toolbar/css/toolbar.css HTTP/1.1" 304 0
INFO 2023-09-24 12:04:08,538 basehttp 20744 18464 "GET /static/debug_toolbar/js/toolbar.js HTTP/1.1" 304 0
INFO 2023-09-24 12:04:08,541 basehttp 20744 18464 "GET /static/debug_toolbar/css/print.css HTTP/1.1" 304 0
INFO 2023-09-24 12:04:08,601 basehttp 20744 18464 "GET /static/debug_toolbar/js/utils.js HTTP/1.1" 304 0
WARNING 2023-09-24 12:04:13,205 log 20744 18464 Not Found: /dz/client
WARNING 2023-09-24 12:04:13,205 basehttp 20744 18464 "GET /dz/client HTTP/1.1" 404 13280
WARNING 2023-09-24 12:04:15,692 log 20744 18464 Not Found: /dz/clients
WARNING 2023-09-24 12:04:15,692 basehttp 20744 18464 "GET /dz/clients HTTP/1.1" 404 13284
WARNING 2023-09-24 12:04:21,003 log 20744 18464 Not Found: /dz/client/2
WARNING 2023-09-24 12:04:21,004 basehttp 20744 18464 "GET /dz/client/2 HTTP/1.1" 404 13288
WARNING 2023-09-24 12:04:23,155 log 20744 18464 Not Found: /dz/client/1
WARNING 2023-09-24 12:04:23,156 basehttp 20744 18464 "GET /dz/client/1 HTTP/1.1" 404 13288
INFO 2023-09-24 12:04:34,914 basehttp 20744 18464 "GET /dz/client/1/7 HTTP/1.1" 301 0
ERROR 2023-09-24 12:04:34,974 log 20744 17572 Internal Server Error: /dz/client/1/7/
Traceback (most recent call last):
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\core\handlers\exception.py", line 55, in inner
    response = get_response(request)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\core\handlers\base.py", line 197, in _get_response
    response = wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\karmaproject\dz_karmaapp\views.py", line 30, in client_orders_7days
    orders = Order.objects.filter(client=client).filter(date__range=[startdate, enddate]).order_by('date_ordered') # ???
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\db\models\manager.py", line 87, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\db\models\query.py", line 1436, in filter
    return self._filter_or_exclude(False, args, kwargs)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\db\models\query.py", line 1454, in _filter_or_exclude
    clone._filter_or_exclude_inplace(negate, args, kwargs)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\db\models\query.py", line 1461, in _filter_or_exclude_inplace
    self._query.add_q(Q(*args, **kwargs))
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\db\models\sql\query.py", line 1545, in add_q
    clause, _ = self._add_q(q_object, self.used_aliases)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\db\models\sql\query.py", line 1576, in _add_q
    child_clause, needed_inner = self.build_filter(
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\db\models\sql\query.py", line 1426, in build_filter
    lookups, parts, reffed_expression = self.solve_lookup_type(arg, summarize)
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\db\models\sql\query.py", line 1236, in solve_lookup_type
    _, field, _, lookup_parts = self.names_to_path(lookup_splitted, self.get_meta())
  File "C:\Users\karmu\Desktop\STUDIUM\5 - Python\dz8\.venv\lib\site-packages\django\db\models\sql\query.py", line 1724, in names_to_path
    raise FieldError(
django.core.exceptions.FieldError: Cannot resolve keyword 'client' into field. Choices are: customer, customer_id, date_ordered, id, products, total_price
ERROR 2023-09-24 12:04:34,999 basehttp 20744 17572 "GET /dz/client/1/7/ HTTP/1.1" 500 126501
INFO 2023-09-24 12:04:41,925 basehttp 20744 17572 "GET /dz/about HTTP/1.1" 301 0
INFO 2023-09-24 12:04:41,943 basehttp 20744 22220 "GET /dz/about/ HTTP/1.1" 200 106
INFO 2023-09-24 12:05:16,086 autoreload 20744 26216 C:\Users\karmu\Desktop\STUDIUM\5 - Python\karmaproject\dz_karmaapp\urls.py changed, reloading.
INFO 2023-09-24 12:05:16,538 autoreload 3816 20820 Watching for file changes with StatReloader
INFO 2023-09-24 12:05:39,338 autoreload 3816 20820 C:\Users\karmu\Desktop\STUDIUM\5 - Python\karmaproject\dz_karmaapp\views.py changed, reloading.
INFO 2023-09-24 12:05:39,738 autoreload 22128 20244 Watching for file changes with StatReloader
INFO 2023-09-24 12:05:51,231 autoreload 22128 20244 C:\Users\karmu\Desktop\STUDIUM\5 - Python\karmaproject\dz_karmaapp\urls.py changed, reloading.
INFO 2023-09-24 12:05:51,653 autoreload 23480 23040 Watching for file changes with StatReloader
INFO 2023-09-24 12:05:59,396 basehttp 23480 25196 "GET /dz/about/ HTTP/1.1" 200 106
INFO 2023-09-24 12:06:02,579 basehttp 23480 25196 "GET /dz/ HTTP/1.1" 200 35
INFO 2023-09-24 12:08:38,015 basehttp 23480 25196 "GET /dz/ HTTP/1.1" 200 35
INFO 2023-09-24 12:08:42,410 basehttp 23480 25196 "GET /dz/about HTTP/1.1" 301 0
INFO 2023-09-24 12:08:42,423 basehttp 23480 20312 "GET /dz/about/ HTTP/1.1" 200 106
INFO 2023-09-24 12:11:05,619 autoreload 23480 23040 C:\Users\karmu\Desktop\STUDIUM\5 - Python\karmaproject\dz_karmaapp\views.py changed, reloading.
INFO 2023-09-24 12:11:06,037 autoreload 12276 19740 Watching for file changes with StatReloader
INFO 2023-09-24 12:13:39,269 autoreload 12276 19740 C:\Users\karmu\Desktop\STUDIUM\5 - Python\karmaproject\dz_karmaapp\views.py changed, reloading.
INFO 2023-09-24 12:13:39,680 autoreload 14520 22828 Watching for file changes with StatReloader
INFO 2023-09-24 12:15:26,646 autoreload 14520 22828 C:\Users\karmu\Desktop\STUDIUM\5 - Python\karmaproject\dz_karmaapp\views.py changed, reloading.
INFO 2023-09-24 12:15:27,049 autoreload 15288 24328 Watching for file changes with StatReloader
INFO 2023-09-24 12:15:32,535 basehttp 15288 23296 "GET /dz/ HTTP/1.1" 200 10115
INFO 2023-09-24 12:15:32,548 basehttp 15288 23296 "GET /static/debug_toolbar/css/toolbar.css HTTP/1.1" 304 0
INFO 2023-09-24 12:15:32,549 basehttp 15288 24136 "GET /static/debug_toolbar/js/toolbar.js HTTP/1.1" 304 0
INFO 2023-09-24 12:15:32,561 basehttp 15288 23296 "GET /static/debug_toolbar/css/print.css HTTP/1.1" 304 0
INFO 2023-09-24 12:15:32,941 basehttp 15288 23296 "GET /static/debug_toolbar/js/utils.js HTTP/1.1" 304 0
INFO 2023-09-24 12:15:59,236 autoreload 15288 24328 C:\Users\karmu\Desktop\STUDIUM\5 - Python\karmaproject\dz_karmaapp\views.py changed, reloading.
INFO 2023-09-24 12:15:59,692 autoreload 23724 22400 Watching for file changes with StatReloader
INFO 2023-09-24 12:16:03,054 basehttp 23724 20192 "GET /dz/about/ HTTP/1.1" 200 10232
INFO 2023-09-24 12:17:04,798 autoreload 23724 22400 C:\Users\karmu\Desktop\STUDIUM\5 - Python\karmaproject\dz_karmaapp\views.py changed, reloading.
INFO 2023-09-24 12:17:05,220 autoreload 10848 19856 Watching for file changes with StatReloader
INFO 2023-09-24 12:17:06,919 basehttp 10848 19592 "GET /dz/about/ HTTP/1.1" 200 10183
INFO 2023-09-24 12:17:13,762 autoreload 10848 19856 C:\Users\karmu\Desktop\STUDIUM\5 - Python\karmaproject\dz_karmaapp\views.py changed, reloading.
INFO 2023-09-24 12:17:14,212 autoreload 25584 5124 Watching for file changes with StatReloader
INFO 2023-09-24 12:17:18,175 basehttp 25584 19876 "GET /dz/about/ HTTP/1.1" 200 10184
INFO 2023-09-24 12:17:23,765 autoreload 25584 5124 C:\Users\karmu\Desktop\STUDIUM\5 - Python\karmaproject\dz_karmaapp\views.py changed, reloading.
INFO 2023-09-24 12:17:24,172 autoreload 22068 10552 Watching for file changes with StatReloader
INFO 2023-09-24 12:17:31,616 autoreload 22068 10552 C:\Users\karmu\Desktop\STUDIUM\5 - Python\karmaproject\dz_karmaapp\views.py changed, reloading.
INFO 2023-09-24 12:17:31,998 autoreload 7552 23764 Watching for file changes with StatReloader
INFO 2023-09-24 12:20:05,443 basehttp 7552 23516 "GET /dz/__debug__/render_panel/?store_id=11c9963bb4b24755b1e2ed3db355d4af&panel_id=HistoryPanel HTTP/1.1" 200 531
INFO 2023-09-24 12:20:11,465 basehttp 7552 23516 "GET /dz/ HTTP/1.1" 200 10114
INFO 2023-09-24 12:20:11,477 basehttp 7552 23516 "GET /static/debug_toolbar/css/toolbar.css HTTP/1.1" 304 0
INFO 2023-09-24 12:20:11,477 basehttp 7552 24164 "GET /static/debug_toolbar/js/toolbar.js HTTP/1.1" 304 0
INFO 2023-09-24 12:20:11,491 basehttp 7552 24164 "GET /static/debug_toolbar/css/print.css HTTP/1.1" 304 0
INFO 2023-09-24 12:20:11,881 basehttp 7552 24164 "GET /static/debug_toolbar/js/utils.js HTTP/1.1" 304 0
INFO 2023-09-24 12:20:12,974 basehttp 7552 24164 "GET /dz/__debug__/render_panel/?store_id=895f786b6dae4e95b1a09cef853b4b3f&panel_id=HistoryPanel HTTP/1.1" 200 2514
INFO 2023-09-24 12:20:12,979 basehttp 7552 24164 "GET /static/debug_toolbar/js/history.js HTTP/1.1" 200 3330
INFO 2023-09-24 12:25:39,465 basehttp 7552 8804 "GET /dz/__debug__/render_panel/?store_id=895f786b6dae4e95b1a09cef853b4b3f&panel_id=TemplatesPanel HTTP/1.1" 200 6005
INFO 2023-09-24 12:25:48,571 basehttp 7552 8804 "GET /dz/__debug__/render_panel/?store_id=895f786b6dae4e95b1a09cef853b4b3f&panel_id=CachePanel HTTP/1.1" 200 1737
INFO 2023-09-24 12:25:50,419 basehttp 7552 8804 "GET /dz/__debug__/render_panel/?store_id=895f786b6dae4e95b1a09cef853b4b3f&panel_id=SignalsPanel HTTP/1.1" 200 2265
INFO 2023-09-24 12:27:09,828 basehttp 7552 8804 "GET /dz/client/add/ HTTP/1.1" 200 11509
INFO 2023-09-24 12:27:09,842 basehttp 7552 7184 "GET /static/debug_toolbar/js/toolbar.js HTTP/1.1" 304 0
INFO 2023-09-24 12:27:09,842 basehttp 7552 8804 "GET /static/debug_toolbar/css/toolbar.css HTTP/1.1" 304 0
INFO 2023-09-24 12:27:09,856 basehttp 7552 8804 "GET /static/debug_toolbar/css/print.css HTTP/1.1" 304 0
INFO 2023-09-24 12:27:10,253 basehttp 7552 8804 "GET /static/debug_toolbar/js/utils.js HTTP/1.1" 304 0
INFO 2023-09-24 12:27:15,393 basehttp 7552 8804 "GET /dz/__debug__/render_panel/?store_id=bdd9db92e14f4381a49f66a80fe0fed5&panel_id=SignalsPanel HTTP/1.1" 200 2310
INFO 2023-09-24 12:27:16,373 basehttp 7552 8804 "GET /dz/__debug__/render_panel/?store_id=bdd9db92e14f4381a49f66a80fe0fed5&panel_id=ProfilingPanel HTTP/1.1" 200 37056
INFO 2023-09-25 07:57:11,243 basehttp 7552 1316 "GET /static/debug_toolbar/css/toolbar.css HTTP/1.1" 304 0
INFO 2023-09-25 07:57:11,244 basehttp 7552 15620 "GET /static/debug_toolbar/js/toolbar.js HTTP/1.1" 304 0
INFO 2023-09-25 07:57:11,252 basehttp 7552 15620 "GET /static/debug_toolbar/css/print.css HTTP/1.1" 304 0
INFO 2023-09-25 07:57:11,660 basehttp 7552 15620 "GET /static/debug_toolbar/js/utils.js HTTP/1.1" 304 0
INFO 2023-09-25 15:55:49,390 basehttp 7552 16460 "GET /static/debug_toolbar/css/toolbar.css HTTP/1.1" 304 0
INFO 2023-09-25 15:55:49,390 basehttp 7552 25604 "GET /static/debug_toolbar/js/toolbar.js HTTP/1.1" 304 0
INFO 2023-09-25 15:55:49,421 basehttp 7552 25604 "GET /static/debug_toolbar/css/print.css HTTP/1.1" 304 0
INFO 2023-09-25 15:55:49,843 basehttp 7552 25604 "GET /static/debug_toolbar/js/utils.js HTTP/1.1" 304 0
